<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Asciidoctor 2.0.26">
<title>Sample and Data Relationship Format for Proteomics (SDRF-Proteomics)</title>
<link rel="stylesheet" href="./css/style.css">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.3/styles/github.min.css">
</head>
<body class="has-doc-header book toc2 toc-left">
<header class="doc-header"><div class="doc-header-brand"><a href="./index.html">SDRF-Proteomics</a></div><nav class="doc-header-nav"><a href="./index.html">Home</a><a href="./specification.html" class="nav-current">Specification</a><a href="./index.html#metadata-guidelines">Metadata Guidelines</a><a href="./index.html#templates">Templates</a><a href="./index.html#tools">Tools</a><a href="./sdrf-explorer.html">Explorer</a><a href="./index.html#contributors">Contributors</a><a href="/dev/" class="version-link">Dev Version</a><a href="https://github.com/bigbio/proteomics-metadata-standard" target="_blank">GitHub</a></nav></header>
<div id="header">
<h1>Sample and Data Relationship Format for Proteomics (SDRF-Proteomics)</h1>
<div id="toc" class="toc2">
<div id="toctitle">Table of Contents</div>
<ul class="sectlevel1">
<li><a href="#status">1. Status of this document</a></li>
<li><a href="#abstract">2. Abstract</a></li>
<li><a href="#motivation">3. Motivation</a></li>
<li><a href="#quick-start">4. Quick Start</a>
<ul class="sectlevel2">
<li><a href="#_minimal_example">4.1. Minimal Example</a></li>
<li><a href="#_key_concepts">4.2. Key Concepts</a></li>
<li><a href="#_format_requirements">4.3. Format Requirements</a></li>
<li><a href="#_scope">4.4. Scope</a></li>
<li><a href="#_getting_started_steps">4.5. Getting Started Steps</a></li>
</ul>
</li>
<li><a href="#validation">5. Validating SDRF Files</a></li>
<li><a href="#_specification_structure">6. Specification structure</a>
<ul class="sectlevel2">
<li><a href="#versioning">6.1. Versioning</a></li>
<li><a href="#notation-conventions">6.2. Notational Conventions</a></li>
<li><a href="#relationship-specifications">6.3. Relationship to other specifications</a></li>
</ul>
</li>
<li><a href="#_sdrf_proteomics_specification">7. SDRF-Proteomics specification</a>
<ul class="sectlevel2">
<li><a href="#sdrf-file-rules">7.1. Format rules</a></li>
<li><a href="#reserved-words">7.2. Reserved words</a></li>
<li><a href="#file-level-metadata">7.3. SDRF file-level metadata</a></li>
<li><a href="#column-headers">7.4. Table Column headers</a></li>
<li><a href="#sdrf-file-standardization">7.5. Table Cell values</a></li>
</ul>
</li>
<li><a href="#from-sample-metadata">8. SDRF-Proteomics: Samples metadata</a>
<ul class="sectlevel2">
<li><a href="#biosamples-integration">8.1. BioSamples database integration</a></li>
<li><a href="#sample-technical-biological-replicates">8.2. Encoding sample technical and biological replicates</a></li>
<li><a href="#pooled-samples">8.3. Pooled samples</a></li>
<li><a href="#_sample_metadata_guidelines">8.4. Sample Metadata Guidelines</a></li>
</ul>
</li>
<li><a href="#from-sample-data">9. SDRF-Proteomics: data files metadata</a>
<ul class="sectlevel2">
<li><a href="#cv-term-format">9.1. CV Term Format for Data File Metadata</a></li>
<li><a href="#sample-prep-fragmentation">9.2. Sample Preparation and Fragmentation (MS-based only)</a></li>
<li><a href="#data-acquisition-method">9.3. Proteomics data acquisition method</a></li>
<li><a href="#_ms_proteomics_template">9.4. MS-Proteomics Template</a></li>
</ul>
</li>
<li><a href="#additional-sdrf-rules">10. Additional SDRF Rules</a>
<ul class="sectlevel2">
<li><a href="#column-cardinality">10.1. Column Cardinality</a></li>
<li><a href="#row-uniqueness">10.2. Row Uniqueness Requirements</a></li>
</ul>
</li>
<li><a href="#templates">11. Templates</a>
<ul class="sectlevel2">
<li><a href="#_what_is_a_template">11.1. What is a Template?</a></li>
<li><a href="#_template_layered_architecture">11.2. Template Layered Architecture</a></li>
<li><a href="#_template_inheritance">11.3. Template Inheritance</a></li>
<li><a href="#_choosing_templates">11.4. Choosing Templates</a></li>
<li><a href="#_core_templates">11.5. Core Templates</a></li>
<li><a href="#experiment-templates">11.6. Experiment-Type Templates</a></li>
<li><a href="#extending-sdrf">11.7. Extending a Template</a>
<ul class="sectlevel3">
<li><a href="#_when_to_add_custom_columns">11.7.1. When to Add Custom Columns</a></li>
<li><a href="#_rules_for_custom_columns">11.7.2. Rules for Custom Columns</a></li>
<li><a href="#_common_additional_columns">11.7.3. Common Additional Columns</a></li>
<li><a href="#_example_adding_study_specific_columns">11.7.4. Example: Adding Study-Specific Columns</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#study-variables">12. Factor Values (Study Variables)</a>
<ul class="sectlevel2">
<li><a href="#_column_format">12.1. Column Format</a></li>
<li><a href="#_when_to_use_factor_values">12.2. When to Use Factor Values</a></li>
<li><a href="#_rules">12.3. Rules</a></li>
<li><a href="#_example">12.4. Example</a></li>
</ul>
</li>
<li><a href="#ontologies-supported">13. Ontologies and Controlled Vocabularies</a></li>
<li><a href="#example-annotated-datasets">14. Examples of Annotated Datasets</a></li>
<li><a href="#_ongoing_template_discussions">15. Ongoing template discussions</a></li>
<li><a href="#_intellectual_property_statement">16. Intellectual Property Statement</a></li>
<li><a href="#_copyright_notice">17. Copyright Notice</a></li>
<li><a href="#_how_to_cite">18. How to cite</a></li>
<li><a href="#_references">References</a></li>
</ul>
</div>
</div>
<div id="content">
<div class="sect1">
<h2 id="status"><a class="anchor" href="#status"></a><a class="link" href="#status">1. Status of this document</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>This document provides information to the proteomics community about a proposed standard for sample metadata annotations in public repositories called Sample and Data Relationship Format (SDRF)-Proteomics. Distribution is unlimited.</p>
</div>
<div class="paragraph">
<p><strong>Version v1.1.0</strong> - 2025-01</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="abstract"><a class="anchor" href="#abstract"></a><a class="link" href="#abstract">2. Abstract</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The Human Proteome Organisation (HUPO) Proteomics Standards Initiative (PSI) defines community standards for data representation in proteomics to facilitate data comparison, exchange, and verification. This document presents a specification for the Sample and Data Relationship Format (SDRF-Proteomics).</p>
</div>
<div class="paragraph">
<p>Further detailed information, including any updates to this document, implementations, and examples is available at <a href="https://github.com/bigbio/proteomics-metadata-standard">SDRF GitHub Repository</a>. The official PSI web page for the document is: <a href="https://www.psidev.info/sdrf-sample-data-relationship-format">HUPO-PSI SDRF</a>.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="motivation"><a class="anchor" href="#motivation"></a><a class="link" href="#motivation">3. Motivation</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>Many resources have emerged that provide raw or integrated proteomics data in the public domain. If these are valuable individually, their integration through re-analysis represents a huge asset for the community <a href="#ref1">[1]</a>.</p>
</div>
<div class="paragraph">
<p>Unfortunately, proteomics experimental design and sample related information are often missing in public repositories or stored in very diverse ways and formats. For example:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>The <a href="https://cptac-data-portal.georgetown.edu/">CPTAC Consortium</a> provides for every dataset a set of Excel files with the information on <a href="https://cptac-data-portal.georgetown.edu/study-summary/S048">each sample</a> including tumor size, origin, but also how every sample is related to a specific raw file (e.g. instrument configuration parameters).</p>
</li>
<li>
<p>As a resource routinely re-analysing public datasets, <a href="https://www.proteomicsdb.org/">ProteomicsDB</a> captures for each sample in the database a minimum number of properties to describe the sample and the related experimental protocol such as tissue, digestion method and instrument.</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Such heterogeneity often prevents data interpretation, reproducibility, and integration of data from different resources. For every proteomics dataset we propose to capture at least three levels of metadata:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>(i) dataset description</p>
</li>
<li>
<p>(ii) the sample metadata and data files acquisition metadata.</p>
</li>
<li>
<p>(iii) The relation between the sample and the data files. The experimental design.</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>The general description includes minimum information to describe the study overall: <a href="http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD016060.0-1&amp;outputMode=XML">title, description, date of publication, type of experiment</a>. In ProteomeXchange partners this metadata is captured at the dataset level, in other omics resources this is captured as IDF file format (e.g. MAGE-TAB). Currently, all ProteomeXchange partners mandate this information for each dataset. However, the information regarding the sample and its relation to the data files (<strong>Figure 1</strong>) is mostly missing <a href="#ref1">[1]</a>.</p>
</div>
<div class="imageblock">
<div class="content">
<img src="https://github.com/bigbio/proteomics-metadata-standard/raw/master/sdrf-proteomics/images/sample-metadata.png" alt="sample metadata">
</div>
</div>
<div class="paragraph">
<p><strong>Figure 1</strong>: SDRF-Proteomics file format stores the information of the sample and its relation to the data files in the dataset. The file format includes not only information about the sample but also about how the data was acquired and processed.</p>
</div>
<div class="paragraph">
<p>Here, we introduced the Sample and Data Relationship Format (SDRF-Proteomics) to capture the sample metadata and its relation to the data files for proteomics experiments. The SDRF-Proteomics format is a tab-delimited file format that describes the sample characteristics and the relationships between samples and data files included in a dataset.</p>
</div>
<div class="paragraph">
<p>This specification, which is a community effort, aims to provide a standard for the proteomics community to annotate the sample metadata and its relation to the data files.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="quick-start"><a class="anchor" href="#quick-start"></a><a class="link" href="#quick-start">4. Quick Start</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>If you&#8217;re new to SDRF-Proteomics, here&#8217;s a minimal example to get you started. An SDRF file is a tab-separated file where each row represents a sample-to-data-file relationship.</p>
</div>
<div class="sect2">
<h3 id="_minimal_example"><a class="anchor" href="#_minimal_example"></a><a class="link" href="#_minimal_example">4.1. Minimal Example</a></h3>
<div class="sdrf-example-table">
<table>
<thead>
<tr>
<th class="sample-col">source name</th>
<th class="sample-col">characteristics[organism]</th>
<th class="sample-col">characteristics[organism part]</th>
<th class="sample-col">characteristics[disease]</th>
<th class="sample-col">characteristics[biological replicate]</th>
<th class="data-col">assay name</th>
<th class="data-col">technology type</th>
<th class="data-col">comment[proteomics data acquisition method]</th>
<th class="data-col">comment[label]</th>
<th class="data-col">comment[instrument]</th>
<th class="data-col">comment[cleavage agent details]</th>
<th class="data-col">comment[fraction identifier]</th>
<th class="data-col">comment[technical replicate]</th>
<th class="data-col">comment[data file]</th>
<th class="factor-col">factor value[disease]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col">sample_1</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">liver</td>
<td class="sample-col">normal</td>
<td class="sample-col">1</td>
<td class="data-col">run_1</td>
<td class="data-col">proteomic profiling by mass spectrometry</td>
<td class="data-col">Data-dependent acquisition</td>
<td class="data-col">label free sample</td>
<td class="data-col">Q Exactive HF</td>
<td class="data-col">NT=Trypsin;AC=MS:1001251</td>
<td class="data-col">1</td>
<td class="data-col">1</td>
<td class="data-col">sample_1.raw</td>
<td class="factor-col">normal</td>
</tr>
<tr>
<td class="sample-col">sample_2</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">liver</td>
<td class="sample-col">hepatocellular carcinoma</td>
<td class="sample-col">1</td>
<td class="data-col">run_2</td>
<td class="data-col">proteomic profiling by mass spectrometry</td>
<td class="data-col">Data-dependent acquisition</td>
<td class="data-col">label free sample</td>
<td class="data-col">Q Exactive HF</td>
<td class="data-col">NT=Trypsin;AC=MS:1001251</td>
<td class="data-col">1</td>
<td class="data-col">1</td>
<td class="data-col">sample_2.raw</td>
<td class="factor-col">hepatocellular carcinoma</td>
</tr>
</tbody>
</table>
<div class="sdrf-legend">
<span class="legend-item"><span class="legend-color sample-bg"></span> Sample metadata (characteristics)</span>
<span class="legend-item"><span class="legend-color data-bg"></span> Data file metadata (comments)</span>
<span class="legend-item"><span class="legend-color factor-bg"></span> Factor values</span>
</div>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
This minimal example shows a <strong>mass spectrometry-based</strong> proteomics experiment. For detailed requirements see the <a href="templates/ms-proteomics.html">MS-Proteomics template</a>. For <strong>affinity-based proteomics</strong> (Olink, SomaScan), different columns are required - see the <a href="templates/affinity-proteomics.html">Affinity-Proteomics template</a>.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="_key_concepts"><a class="anchor" href="#_key_concepts"></a><a class="link" href="#_key_concepts">4.2. Key Concepts</a></h3>
<div class="olist arabic">
<ol class="arabic">
<li>
<p><strong>Sample metadata</strong> uses <code>characteristics[&#8230;&#8203;]</code> columns (e.g., organism, disease)</p>
</li>
<li>
<p><strong>Data file metadata</strong> uses <code>comment[&#8230;&#8203;]</code> columns (e.g., instrument, label)</p>
</li>
<li>
<p><strong>Factor values</strong> use <code>factor value[&#8230;&#8203;]</code> columns to indicate variables under study</p>
</li>
<li>
<p>Each row links one sample to one data file</p>
</li>
</ol>
</div>
</div>
<div class="sect2">
<h3 id="_format_requirements"><a class="anchor" href="#_format_requirements"></a><a class="link" href="#_format_requirements">4.3. Format Requirements</a></h3>
<div class="paragraph">
<p>The SDRF-Proteomics format has the following core requirements:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>The SDRF file is a <strong>tab-delimited format</strong> where each row corresponds to a relationship between a Sample and a Data file.</p>
</li>
<li>
<p>Each column MUST correspond to an attribute/property of the Sample or the Data file.</p>
</li>
<li>
<p>Each cell value MUST be the property value for the corresponding Sample or Data file.</p>
</li>
<li>
<p>The file MUST start with columns describing sample properties (e.g., organism, disease), followed by data file properties (e.g., label, fraction identifier, data file).</p>
</li>
<li>
<p>Unknown values MUST be handled using <a href="#reserved-words">reserved words</a>: <code>not available</code> (value is unknown), <code>not applicable</code> (property doesn&#8217;t apply), or <code>pooled</code> (value is a mixture from multiple samples).</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="_scope"><a class="anchor" href="#_scope"></a><a class="link" href="#_scope">4.4. Scope</a></h3>
<div class="paragraph">
<p>The SDRF-Proteomics format aims to capture the <strong>sample metadata</strong> and its <strong>relationship with data files</strong> (e.g., raw files from mass spectrometers).</p>
</div>
<div class="admonitionblock important">
<table>
<tr>
<td class="icon">
<div class="title">Important</div>
</td>
<td class="content">
SDRF-Proteomics does <strong>not</strong> aim to capture downstream analysis details, including: which samples were compared to which other samples, how samples are combined into study variables, or analysis parameters such as FDR thresholds or p-value cutoffs.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="_getting_started_steps"><a class="anchor" href="#_getting_started_steps"></a><a class="link" href="#_getting_started_steps">4.5. Getting Started Steps</a></h3>
<div class="olist arabic">
<ol class="arabic">
<li>
<p>Choose a <a href="#templates">technology template</a> based on your <a href="#technology-type">technology type</a>: <code>ms-proteomics</code> for <em>proteomic profiling by mass spectrometry</em>, <code>affinity-proteomics</code> for <em>protein expression profiling by antibody/aptamer array</em> (Olink, SomaScan)</p>
</li>
<li>
<p>Add the appropriate <a href="#templates">sample template</a> for your organism (Human, Vertebrates, Invertebrates, Plants, or Cell-lines)</p>
</li>
<li>
<p>Fill in sample metadata (characteristics columns)</p>
</li>
<li>
<p>Fill in data file metadata (comment columns)</p>
</li>
<li>
<p>Add factor values for your experimental variables</p>
</li>
<li>
<p>Validate your file using <a href="https://github.com/bigbio/sdrf-pipelines">sdrf-pipelines</a></p>
</li>
</ol>
</div>
<div class="paragraph">
<p>For detailed guidance, continue reading the full specification below.</p>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="validation"><a class="anchor" href="#validation"></a><a class="link" href="#validation">5. Validating SDRF Files</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The official validator for SDRF-Proteomics files is <strong>sdrf-pipelines</strong>, a Python tool that checks your SDRF file for errors and compliance with the specification.</p>
</div>
<div class="paragraph">
<p>Installation:</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlightjs highlight"><code class="language-bash hljs" data-lang="bash">pip install sdrf-pipelines</code></pre>
</div>
</div>
<div class="paragraph">
<p>Basic Validation:</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlightjs highlight"><code class="language-bash hljs" data-lang="bash"># Validate an SDRF file
parse_sdrf validate-sdrf --sdrf_file your_file.sdrf.tsv

# Validate with a specific template
parse_sdrf validate-sdrf --sdrf_file your_file.sdrf.tsv --template human</code></pre>
</div>
</div>
<div class="paragraph">
<p>For more information, visit: <a href="https://github.com/bigbio/sdrf-pipelines">sdrf-pipelines on GitHub</a></p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_specification_structure"><a class="anchor" href="#_specification_structure"></a><a class="link" href="#_specification_structure">6. Specification structure</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>This document describes the main specification of SDRF-Proteomics, the structure of the specification (<strong>Figure 2</strong>), how to contribute, and extend the specification. SDRF-Proteomics uses a three-tier system for organizing metadata requirements:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="https://github.com/bigbio/proteomics-metadata-standard/blob/master/sdrf-proteomics/README.adoc">The SDRF-Proteomics core specification</a>: This document contains the main specification, requirements and rules for the SDRF-Proteomics format. It also includes the notational conventions and the relationship to other specifications.</p>
</li>
<li>
<p><a href="https://github.com/bigbio/proteomics-metadata-standard/tree/master/sdrf-proteomics/templates">Templates</a>: All templates are organized in the <code>templates/</code> directory. This includes:</p>
<div class="ulist">
<ul>
<li>
<p><strong>Core templates</strong> (organism-based): <a href="templates/human.html">human</a>, <a href="templates/vertebrates.html">vertebrates</a>, <a href="templates/invertebrates.html">invertebrates</a>, <a href="templates/plants.html">plants</a></p>
</li>
<li>
<p><strong>Specialized templates</strong> (experiment-type): <a href="templates/cell-lines.html">cell-lines</a> (sample), <a href="templates/dda-acquisition.html">DDA acquisition</a> (data), <a href="templates/dia-acquisition.html">DIA acquisition</a> (data), <a href="templates/single-cell.html">single-cell</a> (mixed), <a href="templates/affinity-proteomics.html">affinity-proteomics</a> (data), <a href="templates/crosslinking.html">crosslinking</a> (data), <a href="templates/immunopeptidomics.html">immunopeptidomics</a> (mixed), <a href="templates/metaproteomics.html">metaproteomics</a> (mixed)</p>
</li>
</ul>
</div>
</li>
</ul>
</div>
<div class="paragraph">
<p>Each template has its own directory containing a detailed README.adoc with checklists and examples, plus a template file ({name}-template.sdrf.tsv) with column headers. See the <a href="#templates">Templates</a> section.</p>
</div>
<div class="imageblock">
<div class="content">
<img src="images/sdrf-guidelines-structure.png" alt="Logo">
</div>
</div>
<div class="paragraph">
<p><strong>Figure 2</strong>: SDRF-Proteomics specification structure. The main specification defines the core rules and is extended by technology templates (ms-proteomics, affinity-proteomics), sample templates (human, vertebrates, etc.), and specialized experiment-type templates.</p>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
The main specification is in the <code>sdrf-proteomics</code> directory. All templates are organized in <code>sdrf-proteomics/templates/</code>. Templates are extensions of the core specification and should follow all the rules and requirements in the main specification. Each template includes its own detailed documentation with checklists, examples, and annotation guidelines specific to that template type.
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>The official website for SDRF-Proteomics project is <a href="https://github.com/bigbio/proteomics-metadata-standard" class="bare">https://github.com/bigbio/proteomics-metadata-standard</a>. New use cases, changes to the specification and examples can be added by using Pull requests or issues in GitHub to reach the bigbio team.</p>
</div>
<div class="paragraph">
<p>A set of examples and annotated projects from ProteomeXchange can be found here: <a href="https://github.com/bigbio/proteomics-metadata-standard/tree/master/annotated-projects" class="bare">https://github.com/bigbio/proteomics-metadata-standard/tree/master/annotated-projects</a></p>
</div>
<div class="paragraph">
<p>Multiple tools have been implemented to validate, annotate and convert SDRF-Proteomics files. The official validator of SDRF-Proteomics is sdrf-pipelines (Python - <a href="https://github.com/bigbio/sdrf-pipelines" class="bare">https://github.com/bigbio/sdrf-pipelines</a>). This tool allows to validate an SDRF-Proteomics file. In addition, it allows converting SDRF to other popular pipelines and software configure files such as MaxQuant or OpenMS.</p>
</div>
<div class="sect2">
<h3 id="versioning"><a class="anchor" href="#versioning"></a><a class="link" href="#versioning">6.1. Versioning</a></h3>
<div class="paragraph">
<p>The SDRF-Proteomics specification is versioned using the Semantic Versioning 2.0.0 (<a href="https://semver.org/" class="bare">https://semver.org/</a>) scheme. The version number is in the format MAJOR.MINOR.PATCH, where:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>MAJOR version is incremented for incompatible changes to the specification, when major changes are done to the specification.</p>
</li>
<li>
<p>MINOR version is incremented for new features that are backward compatible with the previous version. Guidelines and templates are added or modified.</p>
</li>
<li>
<p>PATCH version is incremented for bug fixes and minor changes that do not affect the specification or the templates. This includes typos, formatting changes, and other minor updates.</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Every change in the specification should be done in GitHub using pull requests into the dev branch. The pull request should include a description of the changes and the reason for the changes. The pull request will be reviewed by the community and merged into the main branch when approved. After the merge, the version number will be updated according to the changes made, the release will be performed, and the Zenodo record will be updated.</p>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
We added the prefix v to the version number to indicate that it is the version of the specification that was used to create the file. Examples: v1.1.0, v2.0.0, v3.0.0.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="notation-conventions"><a class="anchor" href="#notation-conventions"></a><a class="link" href="#notation-conventions">6.2. Notational Conventions</a></h3>
<div class="paragraph">
<p>The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMEND/RECOMMENDED”, “MAY”, “COULD BE”, and “OPTIONAL” are to be interpreted as described in RFC 2119 (<a href="https://www.rfc-editor.org/rfc/rfc2119" class="bare">https://www.rfc-editor.org/rfc/rfc2119</a>).</p>
</div>
</div>
<div class="sect2">
<h3 id="relationship-specifications"><a class="anchor" href="#relationship-specifications"></a><a class="link" href="#relationship-specifications">6.3. Relationship to other specifications</a></h3>
<div class="paragraph">
<p>SDRF-Proteomics is fully compatible with the SDRF file format part of <a href="https://www.ebi.ac.uk/arrayexpress/help/magetab_spec.html">MAGE-TAB</a>. MAGE-TAB is the file format used to store metadata and sample information for transcriptomics experiments. When the ProteomeXchange project file is converted to idf file (project description in MAGE-TAB) and is combined with the SDRF-Proteomics a valid MAGE-TAB is obtained.</p>
</div>
<div class="paragraph">
<p>SDRF-Proteomics sample information can be embedded into mzTab metadata files. The sample metadata in mzTab contains properties as the columns in the SDRF-Proteomics and values as Sample cell values.</p>
</div>
<div class="paragraph">
<p>The SDRF-Proteomics aims to capture the sample metadata and its relationship with the data files (e.g. raw files from mass spectrometers). The SDRF-Proteomics do not aim to capture the downstream analysis part of the experimental design such as what samples should be compared, how they can be combined or parameters for the downstream analysis (FDR or p-values thresholds). The HUPO-PSI community will work in the future to include this information in other file formats such as mzTab or a new type of file format.</p>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_sdrf_proteomics_specification"><a class="anchor" href="#_sdrf_proteomics_specification"></a><a class="link" href="#_sdrf_proteomics_specification">7. SDRF-Proteomics specification</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The SDRF-Proteomics file format describes the sample characteristics and the relationships between samples and data files. The file format is a tab-delimited one where each <strong>ROW</strong> corresponds to a relationship between a Sample and a Data file (in an ms proteomics experiment the data file containing the mass spectra), each <strong>COLUMN</strong> corresponds to an attribute/property of the Sample, the Data file, or the Factor values; and the value in each <strong>CELL</strong> is the specific value of the property for a given Sample/Data file/Factor value (<strong>Figure 3</strong>).</p>
</div>
<div class="sdrf-example-table">
<table>
<thead>
<tr>
<th class="sample-col">source name</th>
<th class="sample-col">characteristics[organism]</th>
<th class="sample-col">characteristics[...]</th>
<th class="sample-col">characteristics[biological replicate]</th>
<th class="data-col">assay name</th>
<th class="data-col">technology type</th>
<th class="data-col">comment[...]</th>
<th class="data-col">comment[data file]</th>
<th class="factor-col">factor value[...]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col">sample_1</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">...</td>
<td class="sample-col">1</td>
<td class="data-col">run_1</td>
<td class="data-col">proteomic profiling by mass spectrometry</td>
<td class="data-col">...</td>
<td class="data-col">sample_1.raw</td>
<td class="factor-col">...</td>
</tr>
<tr>
<td class="sample-col">sample_2</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">...</td>
<td class="sample-col">2</td>
<td class="data-col">run_2</td>
<td class="data-col">proteomic profiling by mass spectrometry</td>
<td class="data-col">...</td>
<td class="data-col">sample_2.raw</td>
<td class="factor-col">...</td>
</tr>
</tbody>
</table>
<div class="sdrf-legend">
<span class="legend-item"><span class="legend-color sample-bg"></span> Sample metadata (characteristics)</span>
<span class="legend-item"><span class="legend-color data-bg"></span> Data file metadata (comments)</span>
<span class="legend-item"><span class="legend-color factor-bg"></span> Factor values</span>
</div>
</div>
<div class="paragraph">
<p><strong>Figure 3</strong>: SDRF-Proteomics in a nutshell. Each <strong>row</strong> links a sample to a data file. <strong>Columns</strong> represent sample properties (characteristics), data file properties (comments), or experimental variables (factor values).</p>
</div>
<div class="paragraph">
<p>The SDRF-Proteomics format contains three main sections:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>The first section contains the <a href="#from-sample-metadata">sample metadata</a>.</p>
</li>
<li>
<p>The second section contains the <a href="#from-sample-data">data file metadata</a>.</p>
</li>
<li>
<p>The third section contains the <a href="#study-variables">factor values</a> properties.</p>
</li>
</ul>
</div>
<div class="sect2">
<h3 id="sdrf-file-rules"><a class="anchor" href="#sdrf-file-rules"></a><a class="link" href="#sdrf-file-rules">7.1. Format rules</a></h3>
<div class="ulist">
<ul>
<li>
<p><strong>Case sensitivity</strong>: Text values are case-insensitive, but <strong>column names are case-sensitive</strong>. Use lowercase for all column names (e.g., <code>source name</code>, <code>characteristics[organism]</code>, <code>comment[label]</code>). Incorrect casing like <code>Source Name</code> or <code>Characteristics[organism]</code> will cause validation failures.</p>
</li>
<li>
<p><strong>Space sensitivity</strong>: The SDRF is sensitive to spaces in column names (<code>sourcename</code> ≠ <code>source name</code>). Column names must include appropriate spaces (e.g., <code>source name</code>, not <code>sourcename</code>) but must NOT have a space before the bracket (e.g., <code>characteristics[organism]</code>, not <code>characteristics [organism]</code>).</p>
</li>
<li>
<p><strong>Column order</strong>: The SDRF columns follows some structure; first the sample metadata columns in <a href="#from-sample-metadata">Chapter 8</a>; then the data file metadata columns in <a href="#from-sample-data">Chapter 9</a>; followed by the factor values columns in <a href="#study-variables">Chapter 12</a>.</p>
</li>
<li>
<p><strong>Extension</strong>: The extension of the SDRF file SHOULD be <strong>sdrf.tsv (preferred)</strong> or .txt.</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="reserved-words"><a class="anchor" href="#reserved-words"></a><a class="link" href="#reserved-words">7.2. Reserved words</a></h3>
<div class="paragraph">
<p>There are general scenarios where cell values cannot be provided with actual data. The following reserved words MUST be used in these cases:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>not available</strong>: In some cases, the column is mandatory in the format, but for some samples the corresponding value is unknown or could not be determined. In those cases, users SHOULD use <strong>not available</strong>.</p>
</li>
<li>
<p><strong>not applicable</strong>: In some cases, the column is mandatory, but for some samples the corresponding value or concept does not apply. In those cases, users SHOULD use <strong>not applicable</strong>.</p>
</li>
<li>
<p><strong>anonymized</strong>: In some cases, the value exists but has been intentionally redacted for privacy protection (e.g., in clinical studies with de-identified patient data). In those cases, users SHOULD use <strong>anonymized</strong>.</p>
</li>
<li>
<p><strong>pooled</strong>: In some cases, the sample is a pool of multiple samples (e.g., TMT reference channels), and the value cannot be represented as a single value. In those cases, users SHOULD use <strong>pooled</strong>.</p>
</li>
</ul>
</div>
<table class="tableblock frame-all grid-all stretch">
<caption class="title">Table 1. Reserved words for SDRF cell values</caption>
<colgroup>
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Term</th>
<th class="tableblock halign-left valign-top">Meaning</th>
<th class="tableblock halign-left valign-top">Example</th>
<th class="tableblock halign-left valign-top">Use Case</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">not available</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Value exists but is unknown or could not be determined</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">characteristics[age] = not available</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Patient age was not recorded in the study</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">not applicable</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Value or concept does not apply to this sample</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">characteristics[age] = not applicable</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Synthetic peptide library has no age</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">anonymized</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Value exists but is redacted for privacy protection</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">characteristics[age] = anonymized</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Clinical study with de-identified patient data</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">pooled</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Value represents a mixture of multiple samples</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">characteristics[biological replicate] = pooled</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">TMT reference channel pooled from multiple replicates</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="file-level-metadata"><a class="anchor" href="#file-level-metadata"></a><a class="link" href="#file-level-metadata">7.3. SDRF file-level metadata</a></h3>
<div class="paragraph">
<p>Since version 1.1.0, SDRF-Proteomics supports optional file-level metadata using header comments at the beginning of the file. These header comments provide information about the SDRF file itself, such as the format version, template used, and validation status. This approach is inspired by other omics formats such as VCF (Variant Call Format) file headers and is fully compatible with pandas and other tabular data processing tools.</p>
</div>
<div class="paragraph">
<p>Header comments MUST:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Start with <code>#</code> (single hash) followed by a key-value pair</p>
</li>
<li>
<p>Appear at the very beginning of the file, before the column header row</p>
</li>
<li>
<p>Use the format <code>#key=value</code></p>
</li>
</ul>
</div>
<div class="paragraph">
<p>The following header fields are supported:</p>
</div>
<table class="tableblock frame-all grid-all stretch requirements-table">
<colgroup>
<col style="width: 18%;">
<col style="width: 35%;">
<col style="width: 20%;">
<col style="width: 14%;">
<col style="width: 13%;">
</colgroup>
<thead>
<tr>
<th>Key</th>
<th>Description</th>
<th>Example</th>
<th>Requirement</th>
<th>Ontology Term</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>file_format</code></td>
<td>Identifier for the file format</td>
<td>SDRF</td>
<td><span class="recommended-badge">RECOMMENDED</span></td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000831">PRIDE:0000831</a></td>
</tr>
<tr>
<td><code>version</code></td>
<td>SDRF-Proteomics specification version used</td>
<td>v1.1.0</td>
<td><span class="recommended-badge">RECOMMENDED</span></td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/ncit/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FNCIT_C25714">NCIT:C25714</a></td>
</tr>
<tr>
<td><code>template</code></td>
<td>Name of the template used</td>
<td>ms-proteomics, human, cell-lines</td>
<td><span class="optional-badge">OPTIONAL</span></td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000832">PRIDE:0000832</a></td>
</tr>
<tr>
<td><code>template_version</code></td>
<td>Version of the template</td>
<td>v1.0.0</td>
<td><span class="optional-badge">OPTIONAL</span></td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000833">PRIDE:0000833</a></td>
</tr>
<tr>
<td><code>source</code></td>
<td>Origin or creator of the file</td>
<td>PRIDE, user-generated</td>
<td><span class="optional-badge">OPTIONAL</span></td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/ncit/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FNCIT_C25683">NCIT:C25683</a></td>
</tr>
<tr>
<td><code>validation_hash</code></td>
<td>Hash from validator certification</td>
<td>sha256:abc123...</td>
<td><span class="optional-badge">OPTIONAL</span></td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000834">PRIDE:0000834</a></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>Example of an SDRF file with header comments (simplified example showing only select columns; see <a href="#templates">Chapter 11</a> for complete required columns):</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlightjs highlight"><code class="language-tsv hljs" data-lang="tsv">#file_format=SDRF
#version=v1.1.0
#template=human
#template_version=v1.0.0
#source=PRIDE
source name	characteristics[organism]	characteristics[organism part]	characteristics[disease]	assay name	comment[data file]
sample_1	homo sapiens	liver	normal	run_1	sample_1.raw</code></pre>
</div>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
Header comments are OPTIONAL. SDRF files without header comments are still valid. When present, header comments provide valuable provenance information and enable tools to handle version-specific features appropriately. Header property names use underscores (e.g., <code>file_format</code>, <code>template_version</code>) rather than spaces to maintain consistency with the tab-delimited nature of SDRF files and avoid ambiguity when parsing.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="column-headers"><a class="anchor" href="#column-headers"></a><a class="link" href="#column-headers">7.4. Table Column headers</a></h3>
<div class="paragraph">
<p>Depending on each section the column headers (property names) will be prefixed with the following prefixes:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><code>characteristics</code>: Sample metadata (e.g. <em>characteristics[organism]</em>)</p>
</li>
<li>
<p><code>comment</code>: Data file metadata (e.g. <em>comment[data file]</em>)</p>
</li>
<li>
<p><code>factor value</code>: Factor values properties (e.g. <em>factor value[disease]</em>)</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Each property name MUST be a valid ontology term or a valid controlled vocabulary term. Each section will have some specific order for column headers.</p>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
A list of all controlled vocabularies and ontologies supported are in the <a href="#ontologies-supported">Chapter 13</a> section. On each section we also provide a list of properties that are supported.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="sdrf-file-standardization"><a class="anchor" href="#sdrf-file-standardization"></a><a class="link" href="#sdrf-file-standardization">7.5. Table Cell values</a></h3>
<div class="paragraph">
<p>The value for each property, (e.g. characteristics, comment, factor value) corresponding to each sample or data file can be represented in multiple ways.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>Free Text (Human readable)</strong>: In the free text representation, the value is provided as text without Ontology support (e.g. colon or providing accession numbers). This is only RECOMMENDED when the text inserted in the table is the exact name of an ontology/CV term in EFO. If the term is not in EFO, other ontologies can be used.</p>
</li>
</ul>
</div>
<div class="sdrf-example-table">
<table>
<thead>
<tr>
<th class="sample-col">source name</th>
<th class="sample-col">characteristics[organism]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col">sample 1</td>
<td class="sample-col">homo sapiens</td>
</tr>
<tr>
<td class="sample-col">sample 2</td>
<td class="sample-col">homo sapiens</td>
</tr>
</tbody>
</table>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>Ontology url (Computer readable)</strong>: Users can provide the corresponding URI (Uniform Resource Identifier) of the ontology/CV term as a value. This is recommended for enriched files where the user does not want to use intermediate tools to map from free text to ontology/CV terms.</p>
</li>
</ul>
</div>
<div class="sdrf-example-table">
<table>
<thead>
<tr>
<th class="sample-col">source name</th>
<th class="sample-col">characteristics[organism]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col">Sample 1</td>
<td class="sample-col">http://purl.obolibrary.org/obo/NCBITaxon_9606</td>
</tr>
<tr>
<td class="sample-col">Sample 2</td>
<td class="sample-col">http://purl.obolibrary.org/obo/NCBITaxon_9606</td>
</tr>
</tbody>
</table>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>Key=value representation (Human and Computer readable)</strong>: The current representation aims to provide a mechanism to represent the complete information of the ontology/CV term including Accession, Name and other additional properties. In the key=value pair representation, the Value of the property is represented as an Object with multiple properties, where the key is one of the properties of the object and the value is the corresponding value for the particular key. An example of key value pairs is post-translational modification (see <a href="templates/ms-proteomics.html#_protein_modifications">Protein Modifications</a>):</p>
<div class="literalblock">
<div class="content">
<pre>NT=Glu-&gt;pyro-Glu;MT=fixed;PP=Anywhere;AC=Unimod:27;TA=E</pre>
</div>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="from-sample-metadata"><a class="anchor" href="#from-sample-metadata"></a><a class="link" href="#from-sample-metadata">8. SDRF-Proteomics: Samples metadata</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The Sample metadata section provides information about the samples of origin and their characteristics. Each sample contains a <em>source name</em> (unique identifier) and a set of <em>characteristics</em> columns. The first column of the file should be the <em>source name</em> and the following columns should be the characteristics of the sample. For example, for any proteomics experiment (human, vertebrate, cell line), the following characteristics should be provided:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>source name</strong>: Unique sample name (it can be present multiple times if the same sample is used several times in the same dataset)</p>
</li>
<li>
<p><strong>characteristics[organism]</strong>: The organism of the Sample of origin. Values MUST come from <a href="https://www.ebi.ac.uk/ols4/ontologies/ncbitaxon">NCBI Taxonomy</a>.</p>
</li>
<li>
<p><strong>characteristics[organism part]</strong>: The part of organism&#8217;s anatomy or substance arising from an organism from which the biomaterial was derived (e.g., liver). Values SHOULD come from <a href="https://www.ebi.ac.uk/ols4/ontologies/uberon">UBERON</a> or <a href="https://www.ebi.ac.uk/ols4/ontologies/bto">BTO</a>.</p>
</li>
<li>
<p><strong>characteristics[disease]</strong>: The disease under study in the Sample. Values SHOULD come from <a href="https://www.ebi.ac.uk/ols4/ontologies/mondo">MONDO</a>, <a href="https://www.ebi.ac.uk/ols4/ontologies/efo">EFO</a>, or <a href="https://www.ebi.ac.uk/ols4/ontologies/doid">DOID</a>. For healthy/control samples, use <code>normal</code> (<a href="https://www.ebi.ac.uk/ols4/ontologies/pato/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPATO_0000461">PATO:0000461</a>) - see <a href="metadata-guidelines/sample-metadata.html#disease-annotation">Disease Annotation Guidelines</a>.</p>
</li>
<li>
<p><strong>characteristics[cell type]</strong>: A cell type is a distinct morphological or functional form of cell (e.g., epithelial, glial). Values SHOULD come from <a href="https://www.ebi.ac.uk/ols4/ontologies/cl">Cell Ontology (CL)</a> or <a href="https://www.ebi.ac.uk/ols4/ontologies/bto">BTO</a>.</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Example:</p>
</div>
<div class="sdrf-example-table">
<table>
<thead>
<tr>
<th class="sample-col">source name</th>
<th class="sample-col">characteristics[organism]</th>
<th class="sample-col">characteristics[organism part]</th>
<th class="sample-col">characteristics[disease]</th>
<th class="sample-col">characteristics[cell type]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col">sample_treat</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">liver</td>
<td class="sample-col">liver cancer</td>
<td class="sample-col">not available</td>
</tr>
<tr>
<td class="sample-col">sample_control</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">liver</td>
<td class="sample-col">liver cancer</td>
<td class="sample-col">not available</td>
</tr>
</tbody>
</table>
<div class="sdrf-legend">
<span class="legend-item"><span class="legend-color sample-bg"></span> Sample metadata</span>
</div>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
Additional characteristics can be added depending on the type of the experiment and sample. The <a href="https://github.com/bigbio/proteomics-metadata-standard/tree/master/sdrf-proteomics/templates">SDRF-Proteomics templates</a> defines a set of templates and checklists of properties that should be provided depending on the proteomics experiment. In the core guidelines and templates, main document of SDRF-Proteomics, we explain the major sample properties for different experiments. However, SDRF-Proteomics can be extended using guidelines for specific experiments.
</td>
</tr>
</table>
</div>
<div class="admonitionblock important">
<table>
<tr>
<td class="icon">
<div class="title">Important</div>
</td>
<td class="content">
Each characteristic name in the column header SHOULD be a CV term from the EFO ontology. For example, the header <code>characteristics[organism]</code> corresponds to the ontology term Organism. However the values could be from EFO or other ontologies. For example, we RECOMMEND to use MONDO for diseases because it has better coverage than EFO. For healthy samples, use <code>normal</code> (PATO:0000461) - see <a href="metadata-guidelines/sample-metadata.html#disease-annotation">Disease Annotation Guidelines</a>.
</td>
</tr>
</table>
</div>
<div class="admonitionblock important">
<table>
<tr>
<td class="icon">
<div class="title">Important</div>
</td>
<td class="content">
Multiple values (columns) for the same <code>characteristics</code> term are allowed in SDRF-Proteomics (see <a href="#column-cardinality">Section 10.1</a>). However, it is RECOMMENDED not to use the same column in the same file. If you have multiple phenotypes, you can specify what it refers to or use another more specific term, e.g., "immunophenotype".
</td>
</tr>
</table>
</div>
<div class="sect2">
<h3 id="biosamples-integration"><a class="anchor" href="#biosamples-integration"></a><a class="link" href="#biosamples-integration">8.1. BioSamples database integration</a></h3>
<div class="paragraph">
<p><a href="https://www.ebi.ac.uk/biosamples/">BioSamples</a> provides persistent identifiers for biological samples that enable cross-database linking <a href="#ref5">[5]</a>. Use the optional <em>characteristics[biosample accession number]</em> column to link samples to BioSamples entries (EBI format: SAMEA*, NCBI format: SAMN*).</p>
</div>
<div class="paragraph">
<p><strong>Column:</strong> <code>characteristics[biosample accession number]</code></p>
</div>
<div class="paragraph">
<p><strong>Examples:</strong> <code>SAMN12345678</code> (NCBI), <code>SAMEA12345678</code> (EBI)</p>
</div>
<div class="paragraph">
<p>Example usage:</p>
</div>
<div class="sdrf-example-table">
<table>
<thead>
<tr>
<th class="sample-col">source name</th>
<th class="sample-col">characteristics[biosample accession number]</th>
<th class="sample-col">characteristics[organism]</th>
<th class="sample-col">characteristics[organism part]</th>
<th class="sample-col">characteristics[disease]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col">sample_001</td>
<td class="sample-col">SAMN12345678</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">liver</td>
<td class="sample-col">liver cancer</td>
</tr>
<tr>
<td class="sample-col">sample_002</td>
<td class="sample-col">SAMN12345679</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">liver</td>
<td class="sample-col">normal</td>
</tr>
<tr>
<td class="sample-col">sample_003</td>
<td class="sample-col">SAMEA12345680</td>
<td class="sample-col">mus musculus</td>
<td class="sample-col">brain</td>
<td class="sample-col">normal</td>
</tr>
</tbody>
</table>
<div class="sdrf-legend">
<span class="legend-item"><span class="legend-color sample-bg"></span> Sample metadata</span>
</div>
</div>
<div class="paragraph">
<p>The BioSamples accession number enables connection to sample resources across databases, linking proteomics data with genomics, transcriptomics, and other omics datasets.</p>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
BioSample accession numbers from NCBI follow the format <code>SAMNxxxxxxxxxx</code> and from EBI follow the format <code>SAMEAxxxxxxxxx</code>, where <code>x</code> represents digits. Either NCBI or EBI BioSample accession numbers can be used depending on where the sample is registered. The <em>characteristics[biosample accession number]</em> column is OPTIONAL, but when available, providing BioSample accession numbers is RECOMMENDED to enhance data integration and reusability. Users must first request BioSample accession numbers from the appropriate service (NCBI or EBI) before including them in their SDRF files.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="sample-technical-biological-replicates"><a class="anchor" href="#sample-technical-biological-replicates"></a><a class="link" href="#sample-technical-biological-replicates">8.2. Encoding sample technical and biological replicates</a></h3>
<div class="paragraph">
<p>SDRF-Proteomics uses two columns to track replicates <a href="#ref4">[4]</a>:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>characteristics[biological replicate]</strong>: Identifies independent biological samples within each experimental condition (factor value group). Replicate numbers are assigned per condition, so if you have 2 cancer samples and 2 healthy samples, both groups would have biological replicates numbered 1 and 2.</p>
</li>
<li>
<p><strong>comment[technical replicate]</strong>: Identifies repeated measurements of the same sample (e.g., multiple LC-MS/MS injections)</p>
</li>
</ul>
</div>
<div class="admonitionblock important">
<table>
<tr>
<td class="icon">
<div class="title">Important</div>
</td>
<td class="content">
Biological replicate numbering restarts for each experimental condition (factor value). For example, in a disease study with factor value[disease], the cancer samples would be numbered 1, 2, 3&#8230;&#8203; and the normal samples would independently be numbered 1, 2, 3&#8230;&#8203; This establishes the relationship between each sample and its experimental condition.
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>The following example shows 2 biological replicates, each with 2 fractions and 2 technical replicates:</p>
</div>
<div class="sdrf-example-table">
<table>
<thead>
<tr>
<th class="sample-col">source name</th>
<th class="sample-col">characteristics[biological replicate]</th>
<th class="data-col">assay name</th>
<th class="data-col">comment[label]</th>
<th class="data-col">comment[fraction identifier]</th>
<th class="data-col">comment[technical replicate]</th>
<th class="data-col">comment[data file]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col">patient_001_sample</td>
<td class="sample-col">1</td>
<td class="data-col">run_01</td>
<td class="data-col">label free sample</td>
<td class="data-col">1</td>
<td class="data-col">1</td>
<td class="data-col">P001_F1_TR1.raw</td>
</tr>
<tr>
<td class="sample-col">patient_001_sample</td>
<td class="sample-col">1</td>
<td class="data-col">run_02</td>
<td class="data-col">label free sample</td>
<td class="data-col">2</td>
<td class="data-col">1</td>
<td class="data-col">P001_F2_TR1.raw</td>
</tr>
<tr>
<td class="sample-col">patient_001_sample</td>
<td class="sample-col">1</td>
<td class="data-col">run_03</td>
<td class="data-col">label free sample</td>
<td class="data-col">1</td>
<td class="data-col">2</td>
<td class="data-col">P001_F1_TR2.raw</td>
</tr>
<tr>
<td class="sample-col">patient_001_sample</td>
<td class="sample-col">1</td>
<td class="data-col">run_04</td>
<td class="data-col">label free sample</td>
<td class="data-col">2</td>
<td class="data-col">2</td>
<td class="data-col">P001_F2_TR2.raw</td>
</tr>
<tr>
<td class="sample-col">patient_002_sample</td>
<td class="sample-col">2</td>
<td class="data-col">run_05</td>
<td class="data-col">label free sample</td>
<td class="data-col">1</td>
<td class="data-col">1</td>
<td class="data-col">P002_F1_TR1.raw</td>
</tr>
<tr>
<td class="sample-col">patient_002_sample</td>
<td class="sample-col">2</td>
<td class="data-col">run_06</td>
<td class="data-col">label free sample</td>
<td class="data-col">2</td>
<td class="data-col">1</td>
<td class="data-col">P002_F2_TR1.raw</td>
</tr>
<tr>
<td class="sample-col">patient_002_sample</td>
<td class="sample-col">2</td>
<td class="data-col">run_07</td>
<td class="data-col">label free sample</td>
<td class="data-col">1</td>
<td class="data-col">2</td>
<td class="data-col">P002_F1_TR2.raw</td>
</tr>
<tr>
<td class="sample-col">patient_002_sample</td>
<td class="sample-col">2</td>
<td class="data-col">run_08</td>
<td class="data-col">label free sample</td>
<td class="data-col">2</td>
<td class="data-col">2</td>
<td class="data-col">P002_F2_TR2.raw</td>
</tr>
</tbody>
</table>
<div class="sdrf-legend">
<span class="legend-item"><span class="legend-color sample-bg"></span> Sample metadata</span>
<span class="legend-item"><span class="legend-color data-bg"></span> Data file metadata</span>
</div>
</div>
<div class="paragraph">
<p>In this example:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>Biological replicates</strong>: <code>patient_001_sample</code> and <code>patient_002_sample</code> are different biological samples (different source names), annotated with <code>characteristics[biological replicate]</code> values 1 and 2</p>
</li>
<li>
<p><strong>Technical replicates</strong>: Each biological sample is measured twice (<code>comment[technical replicate]</code> = 1 and 2)</p>
</li>
<li>
<p><strong>Fractions</strong>: Each technical replicate has 2 fractions (<code>comment[fraction identifier]</code> = 1 and 2)</p>
</li>
</ul>
</div>
<div class="admonitionblock important">
<table>
<tr>
<td class="icon">
<div class="title">Important</div>
</td>
<td class="content">
Both <em>characteristics[biological replicate]</em> and <em>comment[technical replicate]</em> columns are REQUIRED. When no replicates are performed in a study, set both columns to 1 (i.e., each sample is biological replicate 1 and technical replicate 1). For <strong>pooled samples</strong> (e.g., TMT reference channels), use <code>pooled</code> for biological replicate since these samples are mixtures of multiple replicates and assigning a specific replicate number would be misleading.
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>Some examples with explicit annotation of the biological replicates can be found here:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="https://github.com/bigbio/proteomics-metadata-standard/blob/c3a56b076ef381280dfcb0140d2520126ace53ff/annotated-projects/PXD006401/PXD006401.sdrf.tsv" class="bare">https://github.com/bigbio/proteomics-metadata-standard/blob/c3a56b076ef381280dfcb0140d2520126ace53ff/annotated-projects/PXD006401/PXD006401.sdrf.tsv</a></p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="pooled-samples"><a class="anchor" href="#pooled-samples"></a><a class="link" href="#pooled-samples">8.3. Pooled samples</a></h3>
<div class="paragraph">
<p>When multiple samples are pooled into one (e.g., TMT/iTRAQ reference channels for normalization), use the <em>characteristics[pooled sample]</em> column to indicate pooling status. Allowed values:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>not pooled</strong>: Regular individual samples</p>
</li>
<li>
<p><strong>pooled</strong>: Sample is pooled but individual sources are unknown</p>
</li>
<li>
<p><strong>SN=sample1;SN=sample2;&#8230;&#8203;</strong>: Lists source names of pooled samples when known</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Example:</p>
</div>
<div class="sdrf-example-table">
<table>
<thead>
<tr>
<th class="sample-col">source name</th>
<th class="sample-col">characteristics[pooled sample]</th>
<th class="sample-col">characteristics[organism]</th>
<th class="sample-col">characteristics[age]</th>
<th class="data-col">comment[label]</th>
<th class="data-col">comment[data file]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col">sample_1</td>
<td class="sample-col">not pooled</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">45Y</td>
<td class="data-col">TMT126</td>
<td class="data-col">file01.raw</td>
</tr>
<tr>
<td class="sample-col">sample_2</td>
<td class="sample-col">not pooled</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">52Y</td>
<td class="data-col">TMT127N</td>
<td class="data-col">file01.raw</td>
</tr>
<tr>
<td class="sample-col">pooled_ref</td>
<td class="sample-col">SN=sample_1;SN=sample_2</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">pooled</td>
<td class="data-col">TMT131C</td>
<td class="data-col">file01.raw</td>
</tr>
</tbody>
</table>
<div class="sdrf-legend">
<span class="legend-item"><span class="legend-color sample-bg"></span> Sample metadata</span>
<span class="legend-item"><span class="legend-color data-bg"></span> Data file metadata</span>
</div>
</div>
<div class="admonitionblock tip">
<table>
<tr>
<td class="icon">
<div class="title">Tip</div>
</td>
<td class="content">
For pooled samples, use <code>pooled</code> for individual-specific fields (biological replicate, age, sex) to indicate a mixture rather than a single sample.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="_sample_metadata_guidelines"><a class="anchor" href="#_sample_metadata_guidelines"></a><a class="link" href="#_sample_metadata_guidelines">8.4. Sample Metadata Guidelines</a></h3>
<div class="paragraph">
<p>For detailed guidance on annotating sample metadata, refer to the following conventions documents:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="metadata-guidelines/sample-metadata.html">Sample Metadata Guidelines</a> - Detailed guidelines for age, sex, disease, organism part, cell type, developmental stage, spiked-in samples, and other sample characteristics</p>
</li>
<li>
<p><a href="templates/human.html">Human Sample Metadata Guidelines</a> - Human-specific metadata including disease staging, treatment history, demographics, and lifestyle factors</p>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="from-sample-data"><a class="anchor" href="#from-sample-data"></a><a class="link" href="#from-sample-data">9. SDRF-Proteomics: data files metadata</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The connection between samples and data files is done using properties annotated with the <code>comment</code> prefix. All properties referring to a data file (e.g., MS run file) are annotated with the category <code>comment</code>. This differentiates data file properties from sample properties (characteristics).</p>
</div>
<div class="sect2">
<h3 id="cv-term-format"><a class="anchor" href="#cv-term-format"></a><a class="link" href="#cv-term-format">9.1. CV Term Format for Data File Metadata</a></h3>
<div class="paragraph">
<p>For data file metadata (<code>comment</code> columns) that reference ontology terms, use the structured format: <code>NT={term name};AC={accession}</code></p>
</div>
<div class="paragraph">
<p>Examples: <code>NT=HCD;AC=PRIDE:0000590</code>, <code>NT=Orbitrap;AC=MS:1000484</code></p>
</div>
<div class="paragraph">
<p>This format enables automated validation and software extraction from raw files. Sample metadata (characteristics) can use simple term names since they are typically human-annotated.</p>
</div>
<div class="paragraph">
<p>The following properties MUST be provided for each data file in <strong>mass spectrometry-based proteomics</strong> experiments. For <strong>affinity-based proteomics</strong> (Olink, SomaScan), see the <a href="templates/affinity-proteomics.html">Affinity-Proteomics template</a> for different required columns.</p>
</div>
<table class="tableblock frame-all grid-all stretch requirements-table">
<colgroup>
<col style="width: 30%;">
<col style="width: 12%;">
<col style="width: 38%;">
<col style="width: 20%;">
</colgroup>
<thead>
<tr>
<th>Column</th>
<th>Requirement</th>
<th>Description</th>
<th>Ontology</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>assay name</code></td>
<td><span class="required-badge">REQUIRED</span></td>
<td>Unique identifier for an MS run/data file</td>
<td>Free text</td>
</tr>
<tr>
<td><code>technology type</code></td>
<td><span class="required-badge">REQUIRED</span></td>
<td>Technology used to capture the data</td>
<td>Fixed values</td>
</tr>
<tr>
<td><code>comment[proteomics data acquisition method]</code></td>
<td><span class="required-badge">REQUIRED</span></td>
<td>DDA, DIA, PRM, SRM</td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000659">PRIDE:0000659</a></td>
</tr>
<tr>
<td><code>comment[label]</code></td>
<td><span class="required-badge">REQUIRED</span></td>
<td>Label applied to sample (or "label free sample")</td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000093">PRIDE - Labels</a></td>
</tr>
<tr>
<td><code>comment[instrument]</code></td>
<td><span class="required-badge">REQUIRED</span></td>
<td>Mass spectrometer model</td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/ms/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FMS_1000031">PSI-MS - Instruments</a></td>
</tr>
<tr>
<td><code>comment[cleavage agent details]</code></td>
<td><span class="required-badge">REQUIRED</span></td>
<td>Enzyme information (use "not applicable" for top-down/undigested samples)</td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/ms/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FMS_1001045">PSI-MS - Cleavage agents</a></td>
</tr>
<tr>
<td><code>comment[fraction identifier]</code></td>
<td><span class="required-badge">REQUIRED</span></td>
<td>Fraction number (1 if not fractionated)</td>
<td>Integer</td>
</tr>
<tr>
<td><code>comment[technical replicate]</code></td>
<td><span class="required-badge">REQUIRED</span></td>
<td>Technical replicate number (1 if none)</td>
<td>Integer</td>
</tr>
<tr>
<td><code>comment[data file]</code></td>
<td><span class="required-badge">REQUIRED</span></td>
<td>Name of the raw file</td>
<td>Free text</td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>Example:</p>
</div>
<div class="sdrf-example-table">
<table>
<thead>
<tr>
<th class="sample-col">source name</th>
<th class="data-col">assay name</th>
<th class="data-col">technology type</th>
<th class="data-col">comment[proteomics data acquisition method]</th>
<th class="data-col">comment[label]</th>
<th class="data-col">comment[instrument]</th>
<th class="data-col">comment[data file]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col">sample_1</td>
<td class="data-col">sample1_run1</td>
<td class="data-col">proteomic profiling by mass spectrometry</td>
<td class="data-col">data-dependent acquisition</td>
<td class="data-col">label free sample</td>
<td class="data-col">Q Exactive HF</td>
<td class="data-col">sample1.raw</td>
</tr>
</tbody>
</table>
<div class="sdrf-legend">
<span class="legend-item"><span class="legend-color sample-bg"></span> Sample metadata</span>
<span class="legend-item"><span class="legend-color data-bg"></span> Data file metadata</span>
</div>
</div>
</div>
<div class="sect2">
<h3 id="sample-prep-fragmentation"><a class="anchor" href="#sample-prep-fragmentation"></a><a class="link" href="#sample-prep-fragmentation">9.2. Sample Preparation and Fragmentation (MS-based only)</a></h3>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
This section applies to <strong>mass spectrometry-based proteomics</strong> experiments only. For affinity-based proteomics, these properties do not apply.
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>For detailed documentation of sample preparation and MS/MS fragmentation properties, see the <a href="templates/ms-proteomics.html">MS-Proteomics Template</a>:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>Sample preparation</strong>: depletion, reduction reagent, alkylation reagent</p>
</li>
<li>
<p><strong>Fractionation</strong>: fractionation method (used with <code>comment[fraction identifier]</code>)</p>
</li>
<li>
<p><strong>Fragmentation</strong>: collision energy, dissociation method</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="data-acquisition-method"><a class="anchor" href="#data-acquisition-method"></a><a class="link" href="#data-acquisition-method">9.3. Proteomics data acquisition method</a></h3>
<div class="paragraph">
<p>Proteomics data acquisition method can happen in multiple ways: Data Dependent Acquisition (DDA), Data Independent Acquisition (DIA), and targeted approaches. The SDRF-Proteomics file format REQUIRES capturing the method used for the data acquisition in the <em>comment[proteomics data acquisition method]</em> column. The values MUST be children of the PRIDE ontology term <a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000659">proteomics data acquisition method (PRIDE:0000659)</a>. The following values are commonly used:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000627">data-dependent acquisition</a></p>
</li>
<li>
<p><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000450">data-independent acquisition</a></p>
<div class="ulist">
<ul>
<li>
<p><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000650?lang=en">diaPASEF</a></p>
</li>
<li>
<p><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000447">SWATH MS</a></p>
</li>
</ul>
</div>
</li>
<li>
<p><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000629">parallel reaction monitoring</a></p>
</li>
<li>
<p><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000630">selected reaction monitoring</a></p>
</li>
</ul>
</div>
<div class="admonitionblock important">
<table>
<tr>
<td class="icon">
<div class="title">Important</div>
</td>
<td class="content">
The <em>comment[proteomics data acquisition method]</em> column is REQUIRED for all mass spectrometry-based SDRF files. This field must be explicitly specified and cannot be omitted or assumed.
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>You can find an example of a DIA experiment in the following link: <a href="https://github.com/bigbio/proteomics-metadata-standard/blob/master/annotated-projects/PXD018830/PXD018830-DIA.sdrf.tsv">DIA example</a></p>
</div>
<div class="admonitionblock tip">
<table>
<tr>
<td class="icon">
<div class="title">Tip</div>
</td>
<td class="content">
For DIA experiments, additional properties like MS1 scan range can be captured. See <a href="templates/ms-proteomics.html#_dia_scan_window">DIA Scan Window Limits</a> in the MS-Proteomics Template.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="_ms_proteomics_template"><a class="anchor" href="#_ms_proteomics_template"></a><a class="link" href="#_ms_proteomics_template">9.4. MS-Proteomics Template</a></h3>
<div class="paragraph">
<p>For detailed guidance on data file metadata, refer to the conventions document:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="templates/ms-proteomics.html">MS-Proteomics Template</a> - Detailed guidelines for labels, instruments, modifications, cleavage agents, mass tolerances, RAW file URIs, and other data file properties</p>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="additional-sdrf-rules"><a class="anchor" href="#additional-sdrf-rules"></a><a class="link" href="#additional-sdrf-rules">10. Additional SDRF Rules</a></h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="column-cardinality"><a class="anchor" href="#column-cardinality"></a><a class="link" href="#column-cardinality">10.1. Column Cardinality</a></h3>
<div class="paragraph">
<p>Some columns can appear multiple times for the same sample. The cardinality rules are:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>Single (1)</strong>: Column appears exactly once per sample (e.g., <code>characteristics[biological replicate]</code>)</p>
</li>
<li>
<p><strong>Multiple (*)</strong>: Column can appear multiple times (e.g., <code>comment[modification parameters]</code> can specify multiple post-translational modifications)</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Example of multiple <code>comment[modification parameters]</code> columns:</p>
</div>
<div class="sdrf-example-table">
<table>
<thead>
<tr>
<th class="sample-col">source name</th>
<th class="sample-col">characteristics[...]</th>
<th class="data-col">comment[modification parameters]</th>
<th class="data-col">comment[modification parameters]</th>
<th class="data-col">...</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col">sample-1</td>
<td class="sample-col">...</td>
<td class="data-col">NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=fixed;PP=Anywhere</td>
<td class="data-col">NT=Oxidation;AC=UNIMOD:35;TA=M;MT=variable;PP=Anywhere</td>
<td class="data-col">...</td>
</tr>
</tbody>
</table>
<div class="sdrf-legend">
<span class="legend-item"><span class="legend-color sample-bg"></span> Sample metadata</span>
<span class="legend-item"><span class="legend-color data-bg"></span> Data file metadata</span>
</div>
</div>
</div>
<div class="sect2">
<h3 id="row-uniqueness"><a class="anchor" href="#row-uniqueness"></a><a class="link" href="#row-uniqueness">10.2. Row Uniqueness Requirements</a></h3>
<div class="paragraph">
<p>SDRF files must satisfy specific uniqueness constraints to ensure data integrity and enable proper indexing by analysis tools.</p>
</div>
<div class="paragraph">
<p><strong>Error-level constraint (validation fails):</strong> The combination of <code>source name</code> + <code>assay name</code> + <code>comment[label]</code> MUST be unique across all rows in the SDRF file. If two rows have identical values for all three columns, validation will fail with an error. This constraint ensures that each sample-run-label combination can be uniquely identified.</p>
</div>
<div class="paragraph">
<p><strong>Warning-level constraint (validation warns):</strong> The combination of <code>source name</code> + <code>assay name</code> SHOULD be unique across all rows. Non-unique combinations will generate a warning during validation. This constraint helps identify potential issues where the same sample appears to have multiple entries for the same MS run without distinguishing labels.</p>
</div>
<div class="paragraph">
<p><strong>Assay name uniqueness:</strong> Each distinct MS run/data file MUST have exactly one globally unique <code>assay name</code>, and no two different data files may share an assay name. To ensure uniqueness, it is RECOMMENDED to incorporate sample-specific information in assay names, such as sample IDs or replicate numbers (e.g., "sample1_run1", "sample1_run2", "patient001_fraction01").</p>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
For multiplexed experiments (e.g., TMT, iTRAQ), multiple SDRF rows will share the same <code>assay name</code> because multiple samples are analyzed in a single MS run. In these cases, the <code>comment[label]</code> column distinguishes between different samples within the same run, and the combination of <code>source name</code> + <code>assay name</code> + <code>comment[label]</code> remains unique.
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>Example of valid multiplexed experiment:</p>
</div>
<div class="sdrf-example-table">
<table>
<thead>
<tr>
<th class="sample-col">source name</th>
<th>...</th>
<th class="data-col">assay name</th>
<th class="data-col">comment[label]</th>
<th>...</th>
<th class="data-col">comment[data file]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col">sample_A</td>
<td>...</td>
<td class="data-col">TMT_batch1_run1</td>
<td class="data-col">TMT126</td>
<td>...</td>
<td class="data-col">batch1_run1.raw</td>
</tr>
<tr>
<td class="sample-col">sample_B</td>
<td>...</td>
<td class="data-col">TMT_batch1_run1</td>
<td class="data-col">TMT127N</td>
<td>...</td>
<td class="data-col">batch1_run1.raw</td>
</tr>
<tr>
<td class="sample-col">sample_C</td>
<td>...</td>
<td class="data-col">TMT_batch1_run1</td>
<td class="data-col">TMT127C</td>
<td>...</td>
<td class="data-col">batch1_run1.raw</td>
</tr>
<tr>
<td class="sample-col">sample_D</td>
<td>...</td>
<td class="data-col">TMT_batch1_run1</td>
<td class="data-col">TMT128N</td>
<td>...</td>
<td class="data-col">batch1_run1.raw</td>
</tr>
</tbody>
</table>
<div class="sdrf-legend">
<span class="legend-item"><span class="legend-color sample-bg"></span> Sample metadata</span>
<span class="legend-item"><span class="legend-color data-bg"></span> Data file metadata</span>
</div>
</div>
<div class="paragraph">
<p>In this example, all four rows share the same <code>assay name</code> and <code>comment[data file]</code> because they represent different samples multiplexed in a single MS run. The combination of <code>source name</code> + <code>assay name</code> + <code>comment[label]</code> is unique for each row.</p>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="templates"><a class="anchor" href="#templates"></a><a class="link" href="#templates">11. Templates</a></h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="_what_is_a_template"><a class="anchor" href="#_what_is_a_template"></a><a class="link" href="#_what_is_a_template">11.1. What is a Template?</a></h3>
<div class="paragraph">
<p>A <strong>template</strong> in SDRF-Proteomics is a predefined set of metadata columns (both required and recommended) that ensures consistent and complete annotation for a specific type of experiment or sample. Templates serve the same purpose as <strong>metadata checklists</strong>, <strong>minimum information standards</strong> (like MIAPE), or <strong>validation schemas</strong> in other data standards—they define what information must be captured to make a dataset FAIR (Findable, Accessible, Interoperable, Reusable).</p>
</div>
<div class="paragraph">
<p>Each template includes both <strong>sample metadata</strong> (characteristics columns describing the biological sample) and <strong>data file metadata</strong> (comment columns describing the MS data files)—everything needed to create a complete, validated SDRF file.</p>
</div>
</div>
<div class="sect2">
<h3 id="_template_layered_architecture"><a class="anchor" href="#_template_layered_architecture"></a><a class="link" href="#_template_layered_architecture">11.2. Template Layered Architecture</a></h3>
<div class="paragraph">
<p>Templates in SDRF-Proteomics follow a <strong>layered architecture</strong> that separates concerns:</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Layer</th>
<th class="tableblock halign-left valign-top">Purpose</th>
<th class="tableblock halign-left valign-top">Templates</th>
<th class="tableblock halign-left valign-top">Valid Alone?</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>BASE</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Construction artifact with shared columns</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/base.html">base</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">No - never use directly</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>TECHNOLOGY</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Minimum valid templates for data acquisition</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/ms-proteomics.html">ms-proteomics</a>, <a href="templates/affinity-proteomics.html">affinity-proteomics</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Yes</strong> - these are the minimum valid SDRFs</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>SAMPLE</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Organism-specific sample metadata</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/human.html">human</a>, <a href="templates/vertebrates.html">vertebrates</a>, <a href="templates/invertebrates.html">invertebrates</a>, <a href="templates/plants.html">plants</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">No - must combine with TECHNOLOGY</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>EXPERIMENT</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Experiment-specific columns</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/cell-lines.html">cell-lines</a>, <a href="templates/single-cell.html">single-cell</a>, <a href="templates/crosslinking.html">crosslinking</a>, <a href="templates/dda-acquisition.html">dda-acquisition</a>, <a href="templates/dia-acquisition.html">dia-acquisition</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">No - must combine with TECHNOLOGY</p></td>
</tr>
</tbody>
</table>
<div class="admonitionblock tip">
<table>
<tr>
<td class="icon">
<div class="title">Tip</div>
</td>
<td class="content">
The <strong>base</strong> template is an internal construction artifact that defines shared columns inherited by all other templates. It should never be used directly by users. This specification document itself serves as the documentation for base template columns - all columns defined in the <a href="#from-sample-metadata">sample metadata</a> and <a href="#from-sample-data">data file metadata</a> sections are inherited by technology and sample templates.
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p><strong>Key concepts:</strong></p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>Technology templates</strong> (<code>ms-proteomics</code> or <code>affinity-proteomics</code>) are <strong>required</strong> - you must use exactly one</p>
</li>
<li>
<p><strong>Sample templates</strong> (<code>human</code>, <code>vertebrates</code>, etc.) are <strong>recommended</strong> but optional</p>
</li>
<li>
<p><strong>Experiment templates</strong> provide additional methodology-specific columns</p>
</li>
<li>
<p>Templates can be <strong>extended</strong>: add custom columns beyond the template requirements for study-specific metadata</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>When combining templates, include all required columns from each applicable template. The validator will check compliance with all specified templates.</p>
</div>
</div>
<div class="sect2">
<h3 id="_template_inheritance"><a class="anchor" href="#_template_inheritance"></a><a class="link" href="#_template_inheritance">11.3. Template Inheritance</a></h3>
<div class="paragraph">
<p>Templates form a layered inheritance hierarchy. Child templates inherit all columns and validators from parent templates and can add new columns or strengthen requirements.</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlightjs highlight"><code class="language-none hljs">base (CONSTRUCTION ARTIFACT - not valid alone)
  │   Shared columns: source name, organism, organism part,
  │                   biological replicate, assay name, technology type,
  │                   instrument, technical replicate, data file
  │
  ├── ms-proteomics (MINIMUM VALID for MS proteomics)
  │   │   Adds: acquisition method, cleavage agent, label, fraction identifier
  │   │
  │   ├── dda-acquisition (extends ms-proteomics with DDA-specific columns)
  │   ├── dia-acquisition (extends ms-proteomics with DIA-specific columns)
  │   ├── crosslinking (extends ms-proteomics with XL-MS columns)
  │   ├── single-cell (extends ms-proteomics with SCP columns)
  │   └── immunopeptidomics (extends ms-proteomics with MHC-MS columns)
  │
  └── affinity-proteomics (MINIMUM VALID for affinity proteomics)
      │   Adds: instrument, panel name, quantification unit, sample type
      │
      ├── olink (extends affinity-proteomics with Olink-specific columns)
      └── somascan (extends affinity-proteomics with SomaScan-specific columns)

Sample templates (combine with TECHNOLOGY templates):
  ├── human (adds: disease, age, sex, ancestry)
  ├── vertebrates (adds: disease, developmental stage, strain)
  ├── invertebrates (adds: disease, developmental stage, genotype)
  └── plants (adds: disease, developmental stage, growth conditions)

Experiment templates (organism-agnostic):
  └── cell-lines (adds: cell line name, Cellosaurus ID, passage number)</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>Inheritance rules:</strong></p>
</div>
<div class="olist arabic">
<ol class="arabic">
<li>
<p>Child templates inherit all columns from parent templates</p>
</li>
<li>
<p>Child templates may add new columns</p>
</li>
<li>
<p>Child templates may promote <code>optional</code> → <code>recommended</code> → <code>required</code></p>
</li>
<li>
<p>Child templates <strong>must not</strong> weaken requirements (e.g., <code>required</code> → <code>optional</code>)</p>
</li>
<li>
<p>Multiple templates can be combined (e.g., <code>ms-proteomics</code> + <code>human</code> + <code>crosslinking</code>)</p>
</li>
</ol>
</div>
<div class="paragraph">
<p><strong>Column requirement levels in templates:</strong></p>
</div>
<div class="paragraph">
<p>Each column in a template has one of three requirement levels:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>REQUIRED</strong>: The column <strong>MUST</strong> appear in the SDRF file. Validation will fail if this column is missing. Template header files (.sdrf.tsv) always include these columns.</p>
</li>
<li>
<p><strong>RECOMMENDED</strong>: The column <strong>SHOULD</strong> appear in the SDRF file when applicable to the experiment. These columns are included in template header files and validation will generate warnings if missing.</p>
</li>
<li>
<p><strong>OPTIONAL</strong>: The column <strong>MAY</strong> be included if relevant to the study. These columns are NOT included in template header files but are documented in the template checklist. Users can add them as needed.</p>
</li>
</ul>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
When a column is marked as RECOMMENDED or OPTIONAL, it means users should evaluate whether the column applies to their specific experiment. For example, <code>characteristics[cell line]</code> is RECOMMENDED in the immunopeptidomics template because cell lines are commonly used, but tissue samples may use <code>not applicable</code>.
</td>
</tr>
</table>
</div>
<div class="admonitionblock tip">
<table>
<tr>
<td class="icon">
<div class="title">Tip</div>
</td>
<td class="content">
<strong>For developers and maintainers:</strong> For detailed information on YAML template structure, validators, and LinkML schema, see the <a href="metadata-guidelines/template-definitions.html">Template Definitions Guidelines</a>.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="_choosing_templates"><a class="anchor" href="#_choosing_templates"></a><a class="link" href="#_choosing_templates">11.4. Choosing Templates</a></h3>
<div class="paragraph">
<p>First, determine your proteomics technology and select the appropriate technology template:</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Template</th>
<th class="tableblock halign-left valign-top">Use For</th>
<th class="tableblock halign-left valign-top">Documentation</th>
<th class="tableblock halign-left valign-top">Valid Alone?</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong><a href="templates/ms-proteomics.html">MS-Proteomics</a></strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">All mass spectrometry-based proteomics (DDA, DIA, SRM/MRM/PRM)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/ms-proteomics.html">Full documentation</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Yes</strong> - minimum for MS</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong><a href="templates/affinity-proteomics.html">Affinity-Proteomics</a></strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Affinity-based proteomics (Olink, SomaScan)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/affinity-proteomics.html">Full documentation</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Yes</strong> - minimum for affinity</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_core_templates"><a class="anchor" href="#_core_templates"></a><a class="link" href="#_core_templates">11.5. Core Templates</a></h3>
<div class="paragraph">
<p>Add the appropriate sample template based on your sample organism:</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Template</th>
<th class="tableblock halign-left valign-top">Use For</th>
<th class="tableblock halign-left valign-top">Key Additional Columns</th>
<th class="tableblock halign-left valign-top">Documentation</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong><a href="templates/human.html">Human</a></strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Human clinical samples</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">disease, age, sex, ancestry category, individual</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/human.html">Full documentation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong><a href="templates/vertebrates.html">Vertebrates</a></strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Mouse, rat, zebrafish, other vertebrates</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">disease, developmental stage, strain/breed, sex</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/vertebrates.html">Full documentation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong><a href="templates/invertebrates.html">Invertebrates</a></strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Drosophila, C. elegans, insects</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">disease, developmental stage, strain/breed, genotype</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/invertebrates.html">Full documentation</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong><a href="templates/plants.html">Plants</a></strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Arabidopsis, crops, other plants</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">disease, developmental stage, strain/breed, growth conditions, treatment</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/plants.html">Full documentation</a></p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>For detailed explanations of each column, see <a href="#from-sample-metadata">sample metadata</a> for sample properties and <a href="#from-sample-data">data file metadata</a> for data file properties.</p>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
The <a href="metadata-guidelines/sample-metadata.html#cell-type-annotation">characteristics[cell type</a>] column is RECOMMENDED across all templates when the cell type is known or can be determined. Use <code>not available</code> if the cell type cannot be determined (e.g., whole tissue samples, mixed cell populations). For cell line experiments, use the cell-lines template which provides more specific guidance.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="experiment-templates"><a class="anchor" href="#experiment-templates"></a><a class="link" href="#experiment-templates">11.6. Experiment-Type Templates</a></h3>
<div class="paragraph">
<p>In addition to sample templates, SDRF-Proteomics provides specialized experiment-type templates. These templates extend the core templates with methodology-specific columns.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong><a href="templates/dda-acquisition.html">DDA Acquisition</a></strong>: Data-dependent acquisition experiments. Includes columns for dissociation method, collision energy, fractionation method, modification parameters, and mass tolerances.</p>
</li>
<li>
<p><strong><a href="templates/dia-acquisition.html">DIA Acquisition</a></strong>: Data-independent acquisition experiments. Includes columns for scan window limits, isolation window width, DIA method, and spectral library information.</p>
</li>
<li>
<p><strong><a href="templates/cell-lines.html">Cell Lines</a></strong>: Experiments using cell line samples. Includes Cellosaurus integration for cell line identification.</p>
</li>
<li>
<p><strong><a href="templates/single-cell.html">Single-Cell Proteomics</a></strong>: Single-cell proteomics experiments. Includes columns for cell isolation method, carrier proteome, and single-cell identifiers.</p>
</li>
<li>
<p><strong><a href="templates/immunopeptidomics.html">Immunopeptidomics</a></strong>: MHC peptide immunopeptidomics. Includes columns for MHC class, HLA typing, and enrichment methods.</p>
</li>
<li>
<p><strong><a href="templates/crosslinking.html">Crosslinking MS</a></strong>: Cross-linking mass spectrometry experiments. Includes columns for crosslinking reagents and enrichment methods.</p>
</li>
<li>
<p><strong><a href="templates/metaproteomics.html">Metaproteomics</a></strong>: Environmental and microbiome proteomics. Includes columns for environmental sample type and geographic location.</p>
</li>
<li>
<p><strong><a href="templates/affinity-proteomics.html">Affinity Proteomics</a></strong>: Affinity-based proteomics (Olink, SomaScan). Includes columns specific to these platforms.</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>The template files can be downloaded from the <a href="https://github.com/bigbio/proteomics-metadata-standard/tree/master/sdrf-proteomics/templates">templates</a> folder.</p>
</div>
</div>
<div class="sect2">
<h3 id="extending-sdrf"><a class="anchor" href="#extending-sdrf"></a><a class="link" href="#extending-sdrf">11.7. Extending a Template</a></h3>
<div class="paragraph">
<p>Templates define the minimum required and recommended columns for a given experiment type. However, SDRF-Proteomics is designed to be <strong>extensible</strong>:</p>
</div>
<div class="olist arabic">
<ol class="arabic">
<li>
<p><strong>Add experiment-type templates</strong>: If your experiment type has a specialized template (DDA, DIA, cell-lines, etc.), add it to your base configuration. See <a href="#experiment-templates">Experiment-Type Templates</a> above.</p>
</li>
<li>
<p><strong>Add custom columns</strong>: You can add any additional columns to capture study-specific metadata beyond what templates define. This section provides guidance on adding custom columns.</p>
</li>
</ol>
</div>
<div class="sect3">
<h4 id="_when_to_add_custom_columns"><a class="anchor" href="#_when_to_add_custom_columns"></a><a class="link" href="#_when_to_add_custom_columns">11.7.1. When to Add Custom Columns</a></h4>
<div class="paragraph">
<p>Add custom columns when your experiment requires metadata that is:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Not covered by existing templates but important for data interpretation</p>
</li>
<li>
<p>Specific to your experimental design or technology</p>
</li>
<li>
<p>Required for cross-study integration or data reuse</p>
</li>
<li>
<p>Needed to comply with domain-specific standards (e.g., clinical trials, environmental studies)</p>
</li>
</ul>
</div>
</div>
<div class="sect3">
<h4 id="_rules_for_custom_columns"><a class="anchor" href="#_rules_for_custom_columns"></a><a class="link" href="#_rules_for_custom_columns">11.7.2. Rules for Custom Columns</a></h4>
<div class="olist arabic">
<ol class="arabic">
<li>
<p><strong>Use appropriate column prefixes:</strong></p>
<div class="ulist">
<ul>
<li>
<p><code>characteristics[&#8230;&#8203;]</code> - For sample-related metadata (properties of the biological material)</p>
</li>
<li>
<p><code>comment[&#8230;&#8203;]</code> - For technical or protocol-related metadata (data acquisition, processing)</p>
</li>
<li>
<p><code>factor value[&#8230;&#8203;]</code> - For experimental variables (see <a href="#study-variables">Chapter 12</a>)</p>
</li>
</ul>
</div>
</li>
<li>
<p><strong>Follow naming conventions:</strong></p>
<div class="ulist">
<ul>
<li>
<p>Use lowercase for column names inside brackets</p>
</li>
<li>
<p>Use descriptive, specific names (e.g., <code>characteristics[tumor grade]</code> not <code>characteristics[grade]</code>)</p>
</li>
<li>
<p>Use ontology terms when they exist (e.g., <code>characteristics[patient bmi]</code> maps to EFO term)</p>
</li>
</ul>
</div>
</li>
<li>
<p><strong>Use controlled vocabularies when available:</strong></p>
<div class="ulist">
<ul>
<li>
<p>Reference existing ontologies (EFO, MONDO, UBERON, etc.) for values</p>
</li>
<li>
<p>Prefer standardized terms over free text when possible</p>
</li>
<li>
<p>For new terms, follow the key=value format if ontology reference is needed</p>
</li>
</ul>
</div>
</li>
</ol>
</div>
</div>
<div class="sect3">
<h4 id="_common_additional_columns"><a class="anchor" href="#_common_additional_columns"></a><a class="link" href="#_common_additional_columns">11.7.3. Common Additional Columns</a></h4>
<div class="paragraph">
<p>Beyond template requirements, many commonly used columns can enhance your SDRF file. For a comprehensive list of additional columns for sample metadata (treatment, time point, dose, BMI, smoking status, etc.) and data metadata (sample preparation, enrichment method, etc.), see:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="metadata-guidelines/sample-metadata.html#common-additional-columns">Common Additional Columns</a> in the Sample Metadata Guidelines</p>
</li>
<li>
<p><a href="sdrf-terms.html">SDRF Terms Reference</a> for a comprehensive list of commonly used terms</p>
</li>
</ul>
</div>
</div>
<div class="sect3">
<h4 id="_example_adding_study_specific_columns"><a class="anchor" href="#_example_adding_study_specific_columns"></a><a class="link" href="#_example_adding_study_specific_columns">11.7.4. Example: Adding Study-Specific Columns</a></h4>
<div class="paragraph">
<p>For a drug treatment time-course study, you might add columns beyond the template requirements:</p>
</div>
<div class="sdrf-example-table">
<table>
<thead>
<tr>
<th class="sample-col">source name</th>
<th class="sample-col">characteristics[organism]</th>
<th class="sample-col">characteristics[disease]</th>
<th class="sample-col">characteristics[treatment]</th>
<th class="sample-col">characteristics[time point]</th>
<th class="sample-col">characteristics[dose]</th>
<th class="ellipsis-col">...</th>
<th class="factor-col">factor value[treatment]</th>
<th class="factor-col">factor value[time point]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col">sample_ctrl_0h</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">normal</td>
<td class="sample-col">vehicle control</td>
<td class="sample-col">0h</td>
<td class="sample-col">not applicable</td>
<td class="ellipsis-col">...</td>
<td class="factor-col">vehicle control</td>
<td class="factor-col">0h</td>
</tr>
<tr>
<td class="sample-col">sample_drug_24h</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">normal</td>
<td class="sample-col">dexamethasone</td>
<td class="sample-col">24h</td>
<td class="sample-col">100 nM</td>
<td class="ellipsis-col">...</td>
<td class="factor-col">dexamethasone</td>
<td class="factor-col">24h</td>
</tr>
</tbody>
</table>
<div class="sdrf-legend">
<span class="legend-item"><span class="legend-color sample-bg"></span> Sample metadata</span>
<span class="legend-item"><span class="legend-color factor-bg"></span> Factor values</span>
</div>
</div>
<div class="admonitionblock tip">
<table>
<tr>
<td class="icon">
<div class="title">Tip</div>
</td>
<td class="content">
When adding custom columns, check the <a href="metadata-guidelines/sdrf-terms.tsv">SDRF Terms Reference</a> and <a href="https://www.ebi.ac.uk/ols4/">EBI Ontology Lookup Service (OLS)</a> to find existing terms that match your metadata needs. Using standardized terms improves data interoperability.
</td>
</tr>
</table>
</div>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="study-variables"><a class="anchor" href="#study-variables"></a><a class="link" href="#study-variables">12. Factor Values (Study Variables)</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>Factor values identify the experimental variables being studied - the conditions you want to compare in your analysis. They highlight which sample characteristics are the focus of your experiment.</p>
</div>
<div class="sect2">
<h3 id="_column_format"><a class="anchor" href="#_column_format"></a><a class="link" href="#_column_format">12.1. Column Format</a></h3>
<div class="paragraph">
<p><code>factor value[{variable name}]</code></p>
</div>
</div>
<div class="sect2">
<h3 id="_when_to_use_factor_values"><a class="anchor" href="#_when_to_use_factor_values"></a><a class="link" href="#_when_to_use_factor_values">12.2. When to Use Factor Values</a></h3>
<div class="paragraph">
<p>Use factor values to indicate:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>The primary variable(s) under investigation</p>
</li>
<li>
<p>Conditions being compared (e.g., disease vs. normal, treated vs. untreated)</p>
</li>
<li>
<p>Variables that define experimental groups</p>
</li>
</ul>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
Use <code>normal</code> (not "control") in the disease field for healthy samples. "Control" is an experimental design concept, not a disease state. See <a href="metadata-guidelines/sample-metadata.html#disease-annotation">Disease Annotation Guidelines</a> for details.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="_rules"><a class="anchor" href="#_rules"></a><a class="link" href="#_rules">12.3. Rules</a></h3>
<div class="ulist">
<ul>
<li>
<p>Factor value columns SHOULD appear after all characteristics and comment columns</p>
</li>
<li>
<p>Multiple factor values can be used when studying multiple variables</p>
</li>
<li>
<p>The value in a factor value column typically mirrors a characteristics column value</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="_example"><a class="anchor" href="#_example"></a><a class="link" href="#_example">12.4. Example</a></h3>
<div class="paragraph">
<p>In an experiment comparing tumor vs. normal tissue across different cancer stages:</p>
</div>
<div class="sdrf-example-table">
<table>
<thead>
<tr>
<th class="sample-col">source name</th>
<th>...</th>
<th class="sample-col">characteristics[disease]</th>
<th class="sample-col">characteristics[disease staging]</th>
<th>...</th>
<th class="factor-col">factor value[disease]</th>
<th class="factor-col">factor value[disease staging]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col">tumor_sample_1</td>
<td>...</td>
<td class="sample-col">breast carcinoma</td>
<td class="sample-col">stage II</td>
<td>...</td>
<td class="factor-col">breast carcinoma</td>
<td class="factor-col">stage II</td>
</tr>
<tr>
<td class="sample-col">normal_sample_1</td>
<td>...</td>
<td class="sample-col">normal</td>
<td class="sample-col">not applicable</td>
<td>...</td>
<td class="factor-col">normal</td>
<td class="factor-col">not applicable</td>
</tr>
<tr>
<td class="sample-col">tumor_sample_2</td>
<td>...</td>
<td class="sample-col">breast carcinoma</td>
<td class="sample-col">stage III</td>
<td>...</td>
<td class="factor-col">breast carcinoma</td>
<td class="factor-col">stage III</td>
</tr>
</tbody>
</table>
<div class="sdrf-legend">
<span class="legend-item"><span class="legend-color sample-bg"></span> Sample metadata</span>
<span class="legend-item"><span class="legend-color factor-bg"></span> Factor values</span>
</div>
</div>
<div class="paragraph">
<p>In this example, both <code>disease</code> and <code>disease staging</code> are factor values because the experiment aims to compare expression differences between disease states and across cancer stages.</p>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="ontologies-supported"><a class="anchor" href="#ontologies-supported"></a><a class="link" href="#ontologies-supported">13. Ontologies and Controlled Vocabularies</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>SDRF-Proteomics uses ontologies and controlled vocabularies (CVs) to standardize metadata values. The following ontologies are supported:</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 11.1111%;">
<col style="width: 33.3333%;">
<col style="width: 33.3333%;">
<col style="width: 22.2223%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-center valign-top">Category</th>
<th class="tableblock halign-left valign-top">Ontology/CV</th>
<th class="tableblock halign-left valign-top">Description</th>
<th class="tableblock halign-left valign-top">Notes</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-center valign-top" colspan="4"><p class="tableblock"><strong><strong>General Purpose</strong></strong></p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">General</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/efo">Experimental Factor Ontology (EFO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">General experimental metadata</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">General</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/pato">PATO</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Phenotype and Trait Ontology</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">General</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/ncit">NCI Thesaurus (NCIT)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Biomedical terminology</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">General</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/pride">PRIDE Controlled Vocabulary</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Proteomics-specific terms</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top" colspan="4"><p class="tableblock"><strong><strong>Organism and Taxonomy</strong></strong></p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Taxonomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/ncbitaxon">NCBI Taxonomy (NCBITaxon)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Organism classification</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top" colspan="4"><p class="tableblock"><strong><strong>Anatomy and Cell Types</strong></strong></p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/uberon">UBERON</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cross-species anatomy ontology</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Cell Type</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/cl">Cell Ontology (CL)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cell type classification</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/bto">BRENDA Tissue Ontology (BTO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Tissues and cell lines</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/po">Plant Ontology (PO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Plant anatomy and development</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">For plant samples</p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/fbbt">FlyBase Anatomy (FBbt)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Drosophila anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">For Drosophila samples</p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/wbbt">WormBase Anatomy (WBbt)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">C. elegans anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">For C. elegans samples</p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/zfa">Zebrafish Anatomy (ZFA)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Zebrafish anatomy and development</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">For zebrafish samples</p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top" colspan="4"><p class="tableblock"><strong><strong>Disease</strong> (see <a href="metadata-guidelines/sample-metadata.html#disease-annotation">Disease Annotation Guidelines</a>)</strong></p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Disease</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/mondo">Mondo Disease Ontology (MONDO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Unified disease ontology</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">RECOMMENDED</p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Disease</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/efo">Experimental Factor Ontology (EFO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Disease terms from EFO</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Healthy samples</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/pato">Phenotype And Trait Ontology (PATO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Use <code>normal</code> (PATO:0000461) for healthy samples</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top" colspan="4"><p class="tableblock"><strong><strong>Cell Lines</strong></strong></p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Cell Lines</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.cellosaurus.org/">Cellosaurus</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cell line knowledge resource</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">RECOMMENDED</p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Cell Lines</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/clo">Cell Line Ontology (CLO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cell line ontology</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top" colspan="4"><p class="tableblock"><strong><strong>Mass Spectrometry and Proteomics</strong></strong></p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">MS/Proteomics</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/ms">PSI Mass Spectrometry CV (PSI-MS)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Instruments, methods, parameters</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Modifications</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/unimod">Unimod</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Protein modifications database</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Modifications</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/mod">PSI-MOD CV</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Protein modifications ontology</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top" colspan="4"><p class="tableblock"><strong><strong>Other</strong></strong></p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Chemistry</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/chebi">ChEBI</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Chemical Entities of Biological Interest</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Environment</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/envo">Environment Ontology (ENVO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Environmental sample classification</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">For metaproteomics</p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Ancestry</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/hancestro">Human Ancestry Ontology (HANCESTRO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Human ancestry categories</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">For human samples</p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="sect1">
<h2 id="example-annotated-datasets"><a class="anchor" href="#example-annotated-datasets"></a><a class="link" href="#example-annotated-datasets">14. Examples of Annotated Datasets</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The following table provides links to example SDRF files for different experiment types. Click "View in Explorer" to open the SDRF file in the interactive viewer.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Experiment Type</th>
<th class="tableblock halign-left valign-top">Dataset</th>
<th class="tableblock halign-left valign-top">Description</th>
<th class="tableblock halign-left valign-top">View</th>
<th class="tableblock halign-left valign-top">Source</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Label-free</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PXD008934</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Human proteome label-free quantification</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="sdrf-explorer.html?view=PXD008934">View in Explorer</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="./sdrf-explorer.html?view=PXD008934">GitHub</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">TMT</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PXD017710</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">TMT-labeled quantitative proteomics</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="sdrf-explorer.html?view=PXD017710">View in Explorer</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="./sdrf-explorer.html?view=PXD017710">GitHub</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">SILAC</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PXD000612</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SILAC-based quantification</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="sdrf-explorer.html?view=PXD000612">View in Explorer</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="./sdrf-explorer.html?view=PXD000612">GitHub</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">DIA</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PXD018830</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Data-independent acquisition</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="sdrf-explorer.html?view=PXD018830">View in Explorer</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="./sdrf-explorer.html?view=PXD018830">GitHub</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Phosphoproteomics</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PXD000759</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PTM enrichment study</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="sdrf-explorer.html?view=PXD000759">View in Explorer</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="./sdrf-explorer.html?view=PXD000759">GitHub</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cell lines</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PXD001819</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cell line proteomics</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="sdrf-explorer.html?view=PXD001819">View in Explorer</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="./sdrf-explorer.html?view=PXD001819">GitHub</a></p></td>
</tr>
</tbody>
</table>
<div class="admonitionblock tip">
<table>
<tr>
<td class="icon">
<div class="title">Tip</div>
</td>
<td class="content">
Use the <a href="sdrf-explorer.html">SDRF Explorer</a> to browse all {total_datasets}+ annotated datasets with filtering, statistics, and interactive viewing.
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>A comprehensive collection of annotated projects is available at: <a href="https://github.com/bigbio/proteomics-metadata-standard/tree/master/annotated-projects">Annotated Projects Repository</a></p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_ongoing_template_discussions"><a class="anchor" href="#_ongoing_template_discussions"></a><a class="link" href="#_ongoing_template_discussions">15. Ongoing template discussions</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>We have created a file in GitHub <a href="https://github.com/bigbio/proteomics-metadata-standard/blob/master/sdrf-proteomics/templates-under-development.adoc">Ongoing template discussions</a> where we aggregate all the ongoing discussions about the format and new templates.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_intellectual_property_statement"><a class="anchor" href="#_intellectual_property_statement"></a><a class="link" href="#_intellectual_property_statement">16. Intellectual Property Statement</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The PSI takes no position regarding the validity or scope of any intellectual property or other rights that might be claimed to pertain to the implementation or use of the technology described in this document or the extent to which any license under such rights might or might not be available; neither does it represent that it has made any effort to identify any such rights. Copies of claims of rights made available for publication and any assurances of licenses to be made available or the result of an attempt made to obtain a general license or permission for the use of such proprietary rights by implementers or users of this specification can be obtained from the PSI Chair.</p>
</div>
<div class="paragraph">
<p>The PSI invites any interested party to bring to its attention any copyrights, patents or patent applications, or other proprietary rights which may cover technology that may be required to practice this recommendation. Please address the information to the PSI Chair (see contacts information at PSI website).</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_copyright_notice"><a class="anchor" href="#_copyright_notice"></a><a class="link" href="#_copyright_notice">17. Copyright Notice</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>Copyright &#169; Proteomics Standards Initiative (2020). All Rights Reserved.</p>
</div>
<div class="paragraph">
<p>This document and translations of it may be copied and furnished to others, and derivative works that comment on or otherwise explain it or assist in its implementation may be prepared, copied, published, and distributed, in whole or in part, without the restriction of any kind, provided that the above copyright notice and this paragraph are included on all such copies and derivative works. However, this document itself may not be modified in any way, such as by removing the copyright notice or references to the PSI or other organizations, except as needed for the purpose of developing Proteomics Recommendations in which case the procedures for copyrights defined in the PSI Document process must be followed, or as required to translate it into languages other than English.</p>
</div>
<div class="paragraph">
<p>The limited permissions granted above are perpetual and will not be revoked by the PSI or its successors or assigns.</p>
</div>
<div class="paragraph">
<p>This document and the information contained herein is provided on an "AS IS" basis and THE PROTEOMICS STANDARDS INITIATIVE DISCLAIMS ALL WARRANTIES, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO ANY WARRANTY THAT THE USE OF THE INFORMATION HEREIN WILL NOT INFRINGE ANY RIGHTS OR ANY IMPLIED WARRANTIES OF MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE."</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_how_to_cite"><a class="anchor" href="#_how_to_cite"></a><a class="link" href="#_how_to_cite">18. How to cite</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>Please cite this document as:</p>
</div>
<div class="paragraph">
<p>Dai C, Füllgrabe A, Pfeuffer J, Solovyeva EM, Deng J, Moreno P, Kamatchinathan S, Kundu DJ, George N, Fexova S, Grüning B, Föll MC, Griss J, Vaudel M, Audain E, Locard-Paulet M, Turewicz M, Eisenacher M, Uszkoreit J, Van Den Bossche T, Schwämmle V, Webel H, Schulze S, Bouyssié D, Jayaram S, Duggineni VK, Samaras P, Wilhelm M, Choi M, Wang M, Kohlbacher O, Brazma A, Papatheodorou I, Bandeira N, Deutsch EW, Vizcaíno JA, Bai M, Sachsenberg T, Levitsky LI, Perez-Riverol Y. A proteomics sample metadata representation for multiomics integration and big data analysis. Nat Commun. 2021 Oct 6;12(1):5854. doi: 10.1038/s41467-021-26111-3. PMID: 34615866; PMCID: PMC8494749. [Manuscript - <a href="https://www.nature.com/articles/s41467-021-26111-3" class="bare">https://www.nature.com/articles/s41467-021-26111-3</a>]</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_references"><a class="anchor" href="#_references"></a><a class="link" href="#_references">References</a></h2>
<div class="sectionbody">
<div class="ulist bibliography">
<ul class="bibliography">
<li>
<p><a id="ref1"></a>[1] Y. Perez-Riverol, S. European Bioinformatics Community for Mass, Toward a Sample Metadata Standard in Public Proteomics Repositories, J Proteome Res 19(10) (2020) 3906-3909. <a href="https://doi.org/10.1021/acs.jproteome.0c00376">doi:10.1021/acs.jproteome.0c00376</a></p>
</li>
<li>
<p><a id="ref2"></a>[2] A. Gonzalez-Beltran, E. Maguire, S.A. Sansone, P. Rocca-Serra, linkedISA: semantic representation of ISA-Tab experimental metadata, BMC Bioinformatics 15 Suppl 14 (2014) S4. <a href="https://doi.org/10.1186/1471-2105-15-S14-S4">doi:10.1186/1471-2105-15-S14-S4</a></p>
</li>
<li>
<p><a id="ref3"></a>[3] T.F. Rayner, P. Rocca-Serra, P.T. Spellman, et al., A simple spreadsheet-based, MIAME-supportive format for microarray data: MAGE-TAB, BMC Bioinformatics 7 (2006) 489. <a href="https://doi.org/10.1186/1471-2105-7-489">doi:10.1186/1471-2105-7-489</a></p>
</li>
<li>
<p><a id="ref4"></a>[4] P. Blainey, M. Krzywinski, N. Altman, Points of significance: replication, Nat Methods 11(9) (2014) 879-80. <a href="https://doi.org/10.1038/nmeth.3091">doi:10.1038/nmeth.3091</a></p>
</li>
<li>
<p><a id="ref5"></a>[5] D. Gupta, I. Liyanage, Y. Perez-Riverol, et al., BioSamples database: the global hub for sample metadata and multi-omics integration, Nucleic Acids Res (2025). <a href="https://doi.org/10.1093/nar/gkaf1133">doi:10.1093/nar/gkaf1133</a></p>
</li>
</ul>
</div>
</div>
</div>
</div>
<div id="footer">
<div id="footer-text">
Last updated 2026-01-24 08:09:01 UTC
</div>
</div>
<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.3/highlight.min.js"></script>
<script>
if (!hljs.initHighlighting.called) {
  hljs.initHighlighting.called = true
  ;[].slice.call(document.querySelectorAll('pre.highlight > code[data-lang]')).forEach(function (el) { hljs.highlightBlock(el) })
}
</script>
</body>
</html>