<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Asciidoctor 2.0.26">
<title>Sample and Data Relationship Format for Proteomics (SDRF-Proteomics)</title>
<link rel="stylesheet" href="./css/style.css">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.3/styles/github.min.css">
</head>
<body class="has-doc-header book toc2 toc-left">
<header class="doc-header"><div class="doc-header-brand"><a href="./index.html">SDRF-Proteomics</a></div><nav class="doc-header-nav"><a href="./index.html">Home</a><a href="./specification.html" class="nav-current">Specification</a><a href="./index.html#metadata-guidelines">Metadata Guidelines</a><a href="./index.html#templates">Templates</a><a href="./index.html#tools">Tools</a><a href="./sdrf-explorer.html">Explorer</a><a href="./sdrf-editor.html">Editor</a><a href="./index.html#contributors">Contributors</a><a href="/dev/" class="version-link">Dev Version</a><a href="https://github.com/bigbio/proteomics-metadata-standard" target="_blank">GitHub</a></nav></header>
<div id="header">
<h1>Sample and Data Relationship Format for Proteomics (SDRF-Proteomics)</h1>
<div id="toc" class="toc2">
<div id="toctitle">Table of Contents</div>
<ul class="sectlevel1">
<li><a href="#status">1. Status of this document</a></li>
<li><a href="#abstract">2. Abstract</a></li>
<li><a href="#motivation">3. Motivation</a></li>
<li><a href="#_specification_structure">4. Specification structure</a></li>
<li><a href="#quick-start">5. The SDRF-Proteomics Format</a>
<ul class="sectlevel2">
<li><a href="#versioning">5.1. Versioning</a></li>
<li><a href="#sdrf-file-rules">5.2. Format rules</a></li>
<li><a href="#reserved-words">5.3. Reserved words</a></li>
<li><a href="#file-level-metadata">5.4. SDRF file-level metadata</a></li>
<li><a href="#column-headers">5.5. Table Column headers</a></li>
<li><a href="#sdrf-file-standardization">5.6. Table Cell values</a></li>
</ul>
</li>
<li><a href="#validation">6. Validating SDRF Files</a></li>
<li><a href="#from-sample-metadata">7. SDRF-Proteomics: Samples metadata</a>
<ul class="sectlevel2">
<li><a href="#biosamples-integration">7.1. BioSamples database integration</a></li>
<li><a href="#sample-technical-biological-replicates">7.2. Encoding sample technical and biological replicates</a></li>
<li><a href="#pooled-samples">7.3. Pooled samples</a></li>
<li><a href="#_sample_metadata_guidelines">7.4. Sample Metadata Guidelines</a></li>
</ul>
</li>
<li><a href="#from-sample-data">8. SDRF-Proteomics: data files metadata</a>
<ul class="sectlevel2">
<li><a href="#cv-term-format">8.1. CV Term Format for Data File Metadata</a></li>
<li><a href="#sample-prep-fragmentation">8.2. Sample Preparation and Fragmentation (MS-based only)</a></li>
<li><a href="#data-acquisition-method">8.3. Proteomics data acquisition method</a></li>
<li><a href="#_ms_proteomics_template">8.4. MS-Proteomics Template</a></li>
</ul>
</li>
<li><a href="#additional-sdrf-rules">9. Additional SDRF Rules</a>
<ul class="sectlevel2">
<li><a href="#column-cardinality">9.1. Column Cardinality</a></li>
<li><a href="#row-uniqueness">9.2. Row Uniqueness Requirements</a></li>
</ul>
</li>
<li><a href="#templates">10. Templates</a>
<ul class="sectlevel2">
<li><a href="#_template_architecture">10.1. Template Architecture</a></li>
<li><a href="#_specifying_templates_in_sdrf_files">10.2. Specifying Templates in SDRF Files</a></li>
<li><a href="#_available_templates">10.3. Available Templates</a></li>
<li><a href="#_extending_templates">10.4. Extending Templates</a></li>
<li><a href="#_contributing_new_templates">10.5. Contributing New Templates</a></li>
</ul>
</li>
<li><a href="#_factor_values_study_variables">11. Factor Values (Study Variables)</a>
<ul class="sectlevel2">
<li><a href="#_column_format">11.1. Column Format</a></li>
<li><a href="#_when_to_use_factor_values">11.2. When to Use Factor Values</a></li>
<li><a href="#_rules">11.3. Rules</a></li>
<li><a href="#_example">11.4. Example</a></li>
</ul>
</li>
<li><a href="#ontologies-supported">12. Ontologies and Controlled Vocabularies</a></li>
<li><a href="#example-annotated-datasets">13. Examples of Annotated Datasets</a></li>
<li><a href="#_intellectual_property_statement">14. Intellectual Property Statement</a></li>
<li><a href="#_copyright_notice">15. Copyright Notice</a></li>
<li><a href="#_how_to_cite">16. How to cite</a></li>
<li><a href="#_references">References</a></li>
</ul>
</div>
</div>
<div id="content">
<div class="sect1">
<h2 id="status"><a class="anchor" href="#status"></a><a class="link" href="#status">1. Status of this document</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>This document provides information to the proteomics community about a proposed standard for sample metadata annotations in public repositories called Sample and Data Relationship Format (SDRF)-Proteomics. Distribution is unlimited.</p>
</div>
<div class="paragraph">
<p><strong>Version v1.1.0</strong> - 2026-01</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="abstract"><a class="anchor" href="#abstract"></a><a class="link" href="#abstract">2. Abstract</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The Human Proteome Organisation (HUPO) Proteomics Standards Initiative (PSI) defines community standards for data representation in proteomics to facilitate data comparison, exchange, and verification. This document presents a specification for the Sample and Data Relationship Format (SDRF-Proteomics).</p>
</div>
<div class="paragraph">
<p>Further detailed information, including any updates to this document, implementations, and examples is available at <a href="https://github.com/bigbio/proteomics-metadata-standard">SDRF GitHub Repository</a>. The official PSI web page for the document is: <a href="https://www.psidev.info/sdrf-sample-data-relationship-format">HUPO-PSI SDRF</a>.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="motivation"><a class="anchor" href="#motivation"></a><a class="link" href="#motivation">3. Motivation</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>Public proteomics data is valuable, but sample metadata is often missing or stored inconsistently across repositories (e.g., <a href="https://cptac-data-portal.georgetown.edu/">CPTAC</a> uses Excel files, <a href="https://www.proteomicsdb.org/">ProteomicsDB</a> captures minimal properties) <a href="#ref1">[1]</a>. This heterogeneity prevents reproducibility and cross-dataset integration.</p>
</div>
<div class="paragraph">
<p>SDRF-Proteomics addresses this by providing a standard tab-delimited format to capture (<strong>Figure 1</strong>):</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Sample metadata and characteristics</p>
</li>
<li>
<p>Data file acquisition parameters</p>
</li>
<li>
<p>Sample-to-file relationships (experimental design)</p>
</li>
</ul>
</div>
<div class="imageblock">
<div class="content">
<img src="https://github.com/bigbio/proteomics-metadata-standard/raw/master/sdrf-proteomics/images/sample-metadata.png" alt="sample metadata">
</div>
</div>
<div class="paragraph">
<p><strong>Figure 1</strong>: SDRF-Proteomics captures sample information and its relationship to data files.</p>
</div>
<div class="paragraph">
<p>The format is fully compatible with <a href="https://www.ebi.ac.uk/arrayexpress/help/magetab_spec.html">MAGE-TAB</a> SDRF, enabling integration with transcriptomics metadata standards.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_specification_structure"><a class="anchor" href="#_specification_structure"></a><a class="link" href="#_specification_structure">4. Specification structure</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>SDRF-Proteomics uses a two-tier system: this <strong>core specification</strong> defines the format rules, and <strong><a href="#templates">templates</a></strong> provide metadata checklists for specific experiment types (<strong>Figure 2</strong>). Templates are organized in the <a href="https://github.com/bigbio/proteomics-metadata-standard/tree/master/sdrf-proteomics/templates">templates/</a> directory, each with documentation and example files.</p>
</div>
<div class="imageblock">
<div class="content">
<img src="images/sdrf-guidelines-structure.png" alt="Logo">
</div>
</div>
<div class="paragraph">
<p><strong>Figure 2</strong>: SDRF-Proteomics specification structure. The main specification defines the core rules and is extended by sample templates (human, vertebrates, etc.) and experiment-type templates (crosslinking, immunopeptidomics, etc.).</p>
</div>
<div class="paragraph">
<p>The official repository is <a href="https://github.com/bigbio/proteomics-metadata-standard">GitHub</a>, where you can find <a href="https://github.com/bigbio/proteomics-metadata-standard/tree/master/annotated-projects">annotated example projects</a> and the official validator <a href="https://github.com/bigbio/sdrf-pipelines">sdrf-pipelines</a>.</p>
</div>
<div class="admonitionblock important">
<table>
<tr>
<td class="icon">
<div class="title">Important</div>
</td>
<td class="content">
<div class="paragraph">
<p>Throughout this specification, the keywords "MUST", "REQUIRED", "SHOULD", "RECOMMENDED", and "OPTIONAL" are interpreted as described in <a href="https://www.rfc-editor.org/rfc/rfc2119">RFC 2119</a>.</p>
</div>
</td>
</tr>
</table>
</div>
</div>
</div>
<div class="sect1">
<h2 id="quick-start"><a class="anchor" href="#quick-start"></a><a class="link" href="#quick-start">5. The SDRF-Proteomics Format</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>SDRF-Proteomics is a <strong>tab-delimited file</strong> where:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Each <strong>row</strong> = one sample linked to one data file</p>
</li>
<li>
<p>Each <strong>column</strong> = a property (sample characteristic, data file attribute, or factor value)</p>
</li>
<li>
<p>Each <strong>cell</strong> = the property value for that sample/file or a factor value.</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Here&#8217;s a minimal example:</p>
</div>
<div class="sdrf-example-table"><table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 6.6666%;">
<col style="width: 6.6666%;">
<col style="width: 6.6666%;">
<col style="width: 6.6666%;">
<col style="width: 6.6666%;">
<col style="width: 6.6666%;">
<col style="width: 6.6666%;">
<col style="width: 6.6666%;">
<col style="width: 6.6666%;">
<col style="width: 6.6666%;">
<col style="width: 6.6666%;">
<col style="width: 6.6666%;">
<col style="width: 6.6666%;">
<col style="width: 6.6666%;">
<col style="width: 6.6676%;">
</colgroup>
<thead>
<tr>
<th class="sample-col tableblock halign-left valign-top">source name</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[organism]</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[organism part]</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[disease]</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[biological replicate]</th>
<th class="data-col tableblock halign-left valign-top">assay name</th>
<th class="data-col tableblock halign-left valign-top">technology type</th>
<th class="data-col tableblock halign-left valign-top">comment[proteomics data acquisition method]</th>
<th class="data-col tableblock halign-left valign-top">comment[label]</th>
<th class="data-col tableblock halign-left valign-top">comment[instrument]</th>
<th class="data-col tableblock halign-left valign-top">comment[cleavage agent details]</th>
<th class="data-col tableblock halign-left valign-top">comment[fraction identifier]</th>
<th class="data-col tableblock halign-left valign-top">comment[technical replicate]</th>
<th class="data-col tableblock halign-left valign-top">comment[data file]</th>
<th class="factor-col tableblock halign-left valign-top">factor value[disease]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">sample_1</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">homo sapiens</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">liver</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">normal</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">run_1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">proteomic profiling by mass spectrometry</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">Data-dependent acquisition</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">label free sample</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">Q Exactive HF</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">NT=Trypsin;AC=MS:1001251</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">sample_1.raw</p></td>
<td class="factor-col tableblock halign-left valign-top"><p class="tableblock">normal</p></td>
</tr>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">sample_2</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">homo sapiens</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">liver</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">hepatocellular carcinoma</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">run_2</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">proteomic profiling by mass spectrometry</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">Data-dependent acquisition</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">label free sample</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">Q Exactive HF</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">NT=Trypsin;AC=MS:1001251</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">sample_2.raw</p></td>
<td class="factor-col tableblock halign-left valign-top"><p class="tableblock">hepatocellular carcinoma</p></td>
</tr>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">sample_3</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">homo sapiens</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">not available</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">not available</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">run_3</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">proteomic profiling by mass spectrometry</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">Data-dependent acquisition</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">label free sample</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">Q Exactive HF</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">NT=Trypsin;AC=MS:1001251</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">sample_3.raw</p></td>
<td class="factor-col tableblock halign-left valign-top"><p class="tableblock">not available</p></td>
</tr>
</tbody>
</table><div class="sdrf-legend"><div class="legend-item"><span class="legend-color sample"></span>Sample metadata</div><div class="legend-item"><span class="legend-color data"></span>Data file metadata</div><div class="legend-item"><span class="legend-color factor"></span>Factor values</div></div></div>
<div class="paragraph">
<p>The file is organized into three column sections:</p>
</div>
<div class="olist arabic">
<ol class="arabic">
<li>
<p><strong>Sample metadata</strong> (<code>characteristics[&#8230;&#8203;]</code>) - organism, disease, tissue, etc.</p>
</li>
<li>
<p><strong>Data file metadata</strong> (<code>comment[&#8230;&#8203;]</code>) - instrument, label, fraction, data file</p>
</li>
<li>
<p><strong>Factor values</strong> (<code>factor value[&#8230;&#8203;]</code>) - variables under study for statistical analysis</p>
</li>
</ol>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
<div class="ulist">
<ul>
<li>
<p>This example shows <strong>mass spectrometry</strong> proteomics - see <a href="templates/ms-proteomics.html">MS-Proteomics template</a> for full requirements.</p>
</li>
<li>
<p>For <strong>affinity proteomics</strong> (Olink, SomaScan), see <a href="templates/affinity-proteomics.html">Affinity-Proteomics template</a>.</p>
</li>
<li>
<p>Unknown values use <a href="#reserved-words">reserved words</a>: <code>not available</code>, <code>not applicable</code>, or <code>pooled</code>.</p>
</li>
<li>
<p>For a step-by-step tutorial, see the <a href="quickstart.html">Quick Start Guide</a>.</p>
</li>
</ul>
</div>
</td>
</tr>
</table>
</div>
<div class="sect2">
<h3 id="versioning"><a class="anchor" href="#versioning"></a><a class="link" href="#versioning">5.1. Versioning</a></h3>
<div class="paragraph">
<p>The SDRF-Proteomics specification uses <a href="https://semver.org/">Semantic Versioning</a> (MAJOR.MINOR.PATCH). Version numbers are prefixed with "v" (e.g., v1.1.0). Changes are proposed via GitHub pull requests to the dev branch.</p>
</div>
</div>
<div class="sect2">
<h3 id="sdrf-file-rules"><a class="anchor" href="#sdrf-file-rules"></a><a class="link" href="#sdrf-file-rules">5.2. Format rules</a></h3>
<div class="ulist">
<ul>
<li>
<p><strong>Case sensitivity</strong>: Text values are case-insensitive, but <strong>column names are case-sensitive</strong>. Use lowercase for all column names (e.g., <code>source name</code>, <code>characteristics[organism]</code>, <code>comment[label]</code>). Incorrect casing like <code>Source Name</code> or <code>Characteristics[organism]</code> will cause validation failures.</p>
</li>
<li>
<p><strong>Space sensitivity</strong>: The SDRF is sensitive to spaces in column names (<code>sourcename</code> â‰  <code>source name</code>). Column names must include appropriate spaces (e.g., <code>source name</code>, not <code>sourcename</code>) but must NOT have a space before the bracket (e.g., <code>characteristics[organism]</code>, not <code>characteristics [organism]</code>).</p>
</li>
<li>
<p><strong>Column order</strong>: The SDRF columns follows some structure; first the sample metadata columns in <a href="#from-sample-metadata">Chapter 7</a>; then the data file metadata columns in <a href="#from-sample-data">Chapter 8</a>; followed by the factor values columns in <a href="#study-variables">[study-variables]</a>.</p>
</li>
<li>
<p><strong>Extension</strong>: The extension of the SDRF file SHOULD be <strong>sdrf.tsv (preferred)</strong> or .txt.</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="reserved-words"><a class="anchor" href="#reserved-words"></a><a class="link" href="#reserved-words">5.3. Reserved words</a></h3>
<div class="paragraph">
<p>There are general scenarios where cell values cannot be provided with actual data. The following reserved words MUST be used in these cases:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>not available</strong>: In some cases, the column is mandatory in the format, but for some samples the corresponding value is unknown or could not be determined. In those cases, users SHOULD use <strong>not available</strong>.</p>
</li>
<li>
<p><strong>not applicable</strong>: In some cases, the column is mandatory, but for some samples the corresponding value or concept does not apply. In those cases, users SHOULD use <strong>not applicable</strong>.</p>
</li>
<li>
<p><strong>anonymized</strong>: In some cases, the value exists but has been intentionally redacted for privacy protection (e.g., in clinical studies with de-identified patient data). In those cases, users SHOULD use <strong>anonymized</strong>.</p>
</li>
<li>
<p><strong>pooled</strong>: In some cases, the sample is a pool of multiple samples (e.g., TMT reference channels), and the value cannot be represented as a single value. In those cases, users SHOULD use <strong>pooled</strong>.</p>
</li>
</ul>
</div>
<table class="tableblock frame-all grid-all stretch">
<caption class="title">Table 1. Reserved words for SDRF cell values</caption>
<colgroup>
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Term</th>
<th class="tableblock halign-left valign-top">Meaning</th>
<th class="tableblock halign-left valign-top">Example</th>
<th class="tableblock halign-left valign-top">Use Case</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">not available</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Value exists but is unknown or could not be determined</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">characteristics[age] = not available</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Patient age was not recorded in the study</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">not applicable</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Value or concept does not apply to this sample</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">characteristics[age] = not applicable</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Synthetic peptide library has no age</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">anonymized</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Value exists but is redacted for privacy protection</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">characteristics[age] = anonymized</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Clinical study with de-identified patient data</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">pooled</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Value represents a mixture of multiple samples</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">characteristics[biological replicate] = pooled</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">TMT reference channel pooled from multiple replicates</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="file-level-metadata"><a class="anchor" href="#file-level-metadata"></a><a class="link" href="#file-level-metadata">5.4. SDRF file-level metadata</a></h3>
<div class="paragraph">
<p>Since version 1.1.0, SDRF-Proteomics supports file-level metadata using dedicated columns. These columns provide information about the SDRF file itself, such as the specification version, template(s) used, annotation tool, and validation status. This column-based approach maintains compatibility with spreadsheet applications (Excel, Google Sheets) and existing data processing tools.</p>
</div>
<div class="paragraph">
<p>The following metadata columns are supported:</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 18.1818%;">
<col style="width: 36.3636%;">
<col style="width: 18.1818%;">
<col style="width: 9.0909%;">
<col style="width: 18.1819%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Column</th>
<th class="tableblock halign-left valign-top">Description</th>
<th class="tableblock halign-left valign-top">Example Value</th>
<th class="tableblock halign-left valign-top">Requirement</th>
<th class="tableblock halign-left valign-top">Ontology Term</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>comment[sdrf version]</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SDRF-Proteomics specification version used. Should follow semantic versioning format (vMAJOR.MINOR.PATCH)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">v1.1.0</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">RECOMMENDED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PRIDE:0000839</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>comment[sdrf template]</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Template name and version used for annotation. Two formats are supported: simple format (<code>name vX.Y.Z</code>) or key=value format (<code>NT=name;VV=vX.Y.Z</code>). Multiple templates can be specified using multiple columns.</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">human v1.1.0 or NT=human;VV=v1.1.0</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">OPTIONAL</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PRIDE:0000832</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>comment[sdrf annotation tool]</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Software tool, script, or method used to generate or annotate the SDRF file. Two formats are supported: simple format (<code>name vX.Y.Z</code>) or key=value format (<code>NT=name;VV=vX.Y.Z</code>).</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">lesSDRF v0.1.0 or NT=lesSDRF;VV=v0.1.0</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">OPTIONAL</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PRIDE:0000840</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>comment[sdrf validation hash]</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cryptographic hash (e.g., SHA-256) generated after successful validation</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">sha256:abc123&#8230;&#8203;</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">OPTIONAL</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PRIDE:0000834</p></td>
</tr>
</tbody>
</table>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
When combining multiple templates (e.g., <code>human</code> + <code>ms-proteomics</code>), use multiple <code>comment[sdrf template]</code> columns, one per template. The value in each row should be identical for all samples in the file.
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>Example of an SDRF file with metadata columns (simplified example showing only select columns; see <a href="#templates">Chapter 10</a> for complete required columns):</p>
</div>
<div class="sdrf-example-table">
<table>
<thead>
<tr>
<th class="sample-col">source name</th>
<th class="sample-col">characteristics[organism]</th>
<th class="sample-col">characteristics[disease]</th>
<th class="data-col">assay name</th>
<th class="data-col">comment[data file]</th>
<th class="data-col">comment[sdrf version]</th>
<th class="data-col">comment[sdrf template]</th>
<th class="data-col">comment[sdrf template]</th>
<th class="data-col">comment[sdrf annotation tool]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col">sample_1</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">normal</td>
<td class="data-col">run_1</td>
<td class="data-col">sample_1.raw</td>
<td class="data-col">v1.1.0</td>
<td class="data-col">human v1.1.0</td>
<td class="data-col">ms-proteomics v1.1.0</td>
<td class="data-col">lesSDRF v0.1.0</td>
</tr>
<tr>
<td class="sample-col">sample_2</td>
<td class="sample-col">homo sapiens</td>
<td class="sample-col">breast cancer</td>
<td class="data-col">run_2</td>
<td class="data-col">sample_2.raw</td>
<td class="data-col">v1.1.0</td>
<td class="data-col">human v1.1.0</td>
<td class="data-col">ms-proteomics v1.1.0</td>
<td class="data-col">lesSDRF v0.1.0</td>
</tr>
</tbody>
</table>
<div class="sdrf-legend">
<span class="legend-item"><span class="legend-color sample-bg"></span> Sample metadata</span>
<span class="legend-item"><span class="legend-color data-bg"></span> Data file metadata</span>
</div>
</div>
<div class="admonitionblock tip">
<table>
<tr>
<td class="icon">
<div class="title">Tip</div>
</td>
<td class="content">
For backward compatibility, some tools may still support the older header comment format (<code>#key=value</code>). However, the column-based format is the recommended approach for new SDRF files as it is easier to edit in spreadsheet applications and supports per-sample provenance tracking when merging files.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="column-headers"><a class="anchor" href="#column-headers"></a><a class="link" href="#column-headers">5.5. Table Column headers</a></h3>
<div class="paragraph">
<p>Depending on each section the column headers (property names) will be prefixed with the following prefixes:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><code>characteristics</code>: Sample metadata (e.g. <em>characteristics[organism]</em>)</p>
</li>
<li>
<p><code>comment</code>: Data file metadata (e.g. <em>comment[data file]</em>)</p>
</li>
<li>
<p><code>factor value</code>: Factor values properties (e.g. <em>factor value[disease]</em>)</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Each property name MUST be a valid ontology term or a valid controlled vocabulary term. Each section will have some specific order for column headers.</p>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
A list of all controlled vocabularies and ontologies supported are in the <a href="#ontologies-supported">Chapter 12</a> section. On each section we also provide a list of properties that are supported.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="sdrf-file-standardization"><a class="anchor" href="#sdrf-file-standardization"></a><a class="link" href="#sdrf-file-standardization">5.6. Table Cell values</a></h3>
<div class="paragraph">
<p>The value for each property, (e.g. characteristics, comment, factor value) corresponding to each sample or data file can be represented in multiple ways.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>Free Text (Human readable)</strong>: In the free text representation, the value is provided as text without Ontology support (e.g. colon or providing accession numbers). This is only RECOMMENDED when the text inserted in the table is the exact name of an ontology/CV term in EFO. If the term is not in EFO, other ontologies can be used.</p>
</li>
</ul>
</div>
<div class="sdrf-example-table"><table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="sample-col tableblock halign-left valign-top">source name</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[organism]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">sample 1</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">homo sapiens</p></td>
</tr>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">sample 2</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">homo sapiens</p></td>
</tr>
</tbody>
</table><div class="sdrf-legend"><div class="legend-item"><span class="legend-color sample"></span>Sample metadata</div></div></div>
<div class="ulist">
<ul>
<li>
<p><strong>Ontology url (Computer readable)</strong>: Users can provide the corresponding URI (Uniform Resource Identifier) of the ontology/CV term as a value. This is recommended for enriched files where the user does not want to use intermediate tools to map from free text to ontology/CV terms.</p>
</li>
<li>
<p><strong>Key=value representation (Human and Computer readable)</strong>: The current representation aims to provide a mechanism to represent the complete information of the ontology/CV term including Accession, Name and other additional properties. In the key=value pair representation, the Value of the property is represented as an Object with multiple properties, where the key is one of the properties of the object and the value is the corresponding value for the particular key. An example of key value pairs is post-translational modification (see <a href="templates/ms-proteomics.html#_protein_modifications">Protein Modifications</a>):</p>
<div class="literalblock">
<div class="content">
<pre>NT=Glu-&gt;pyro-Glu;MT=fixed;PP=Anywhere;AC=Unimod:27;TA=E</pre>
</div>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="validation"><a class="anchor" href="#validation"></a><a class="link" href="#validation">6. Validating SDRF Files</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The official validator for SDRF-Proteomics files is <a href="https://github.com/bigbio/sdrf-pipelines"><strong>sdrf-pipelines</strong></a>, a Python tool that checks your SDRF file for errors and compliance with the specification.</p>
</div>
<div class="paragraph">
<p>Installation:</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlightjs highlight"><code class="language-bash hljs" data-lang="bash">pip install sdrf-pipelines</code></pre>
</div>
</div>
<div class="paragraph">
<p>Basic Validation:</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlightjs highlight"><code class="language-bash hljs" data-lang="bash"># Validate an SDRF file
parse_sdrf validate-sdrf --sdrf_file your_file.sdrf.tsv

# Validate with a specific template
parse_sdrf validate-sdrf --sdrf_file your_file.sdrf.tsv --template human</code></pre>
</div>
</div>
<div class="paragraph">
<p>For more information, visit: <a href="https://github.com/bigbio/sdrf-pipelines">sdrf-pipelines on GitHub</a></p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="from-sample-metadata"><a class="anchor" href="#from-sample-metadata"></a><a class="link" href="#from-sample-metadata">7. SDRF-Proteomics: Samples metadata</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The Sample metadata section provides information about the samples of origin and their characteristics. Each sample contains a <em>source name</em> (unique identifier) and a set of <em>characteristics</em> columns. The first column of the file should be the <em>source name</em> and the following columns should be the characteristics of the sample. For example, for any proteomics experiment (human, vertebrate, cell line), the following characteristics should be provided:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>source name</strong>: Unique sample name (it can be present multiple times if the same sample is used several times in the same dataset)</p>
</li>
<li>
<p><strong>characteristics[organism]</strong>: The organism of the Sample of origin. Values MUST come from <a href="https://www.ebi.ac.uk/ols4/ontologies/ncbitaxon">NCBI Taxonomy</a>.</p>
</li>
<li>
<p><strong>characteristics[organism part]</strong>: The part of organism&#8217;s anatomy or substance arising from an organism from which the biomaterial was derived (e.g., liver). Values SHOULD come from <a href="https://www.ebi.ac.uk/ols4/ontologies/uberon">UBERON</a> or <a href="https://www.ebi.ac.uk/ols4/ontologies/bto">BTO</a>.</p>
</li>
<li>
<p><strong>characteristics[disease]</strong>: The disease under study in the Sample. Values SHOULD come from <a href="https://www.ebi.ac.uk/ols4/ontologies/mondo">MONDO</a>, <a href="https://www.ebi.ac.uk/ols4/ontologies/efo">EFO</a>, or <a href="https://www.ebi.ac.uk/ols4/ontologies/doid">DOID</a>. For healthy/control samples, use <code>normal</code> (<a href="https://www.ebi.ac.uk/ols4/ontologies/pato/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPATO_0000461">PATO:0000461</a>) - see <a href="metadata-guidelines/sample-metadata.html#disease-annotation">Disease Annotation Guidelines</a>.</p>
</li>
<li>
<p><strong>characteristics[cell type]</strong>: A cell type is a distinct morphological or functional form of cell (e.g., epithelial, glial). Values SHOULD come from <a href="https://www.ebi.ac.uk/ols4/ontologies/cl">Cell Ontology (CL)</a> or <a href="https://www.ebi.ac.uk/ols4/ontologies/bto">BTO</a>.</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Example:</p>
</div>
<div class="sdrf-example-table"><table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
</colgroup>
<thead>
<tr>
<th class="sample-col tableblock halign-left valign-top">source name</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[organism]</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[organism part]</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[disease]</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[cell type]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">sample_treat</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">homo sapiens</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">liver</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">liver cancer</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">not available</p></td>
</tr>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">sample_control</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">homo sapiens</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">liver</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">liver cancer</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">not available</p></td>
</tr>
</tbody>
</table><div class="sdrf-legend"><div class="legend-item"><span class="legend-color sample"></span>Sample metadata</div></div></div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
<div class="ulist">
<ul>
<li>
<p>Additional characteristics can be added per experiment type - see <a href="https://github.com/bigbio/proteomics-metadata-standard/tree/master/sdrf-proteomics/templates">SDRF-Proteomics templates</a> for required properties.</p>
</li>
<li>
<p>Column headers SHOULD use EFO ontology terms (e.g., <code>characteristics[organism]</code>) - see <a href="metadata-guidelines/sample-metadata.html#disease-annotation">Disease Annotation Guidelines</a>.</p>
</li>
<li>
<p>Multiple columns with the same <code>characteristics</code> term are allowed (see <a href="#column-cardinality">Section 9.1</a>), but RECOMMENDED to use more specific terms (e.g., "immunophenotype" instead of duplicate "phenotype").</p>
</li>
</ul>
</div>
</td>
</tr>
</table>
</div>
<div class="sect2">
<h3 id="biosamples-integration"><a class="anchor" href="#biosamples-integration"></a><a class="link" href="#biosamples-integration">7.1. BioSamples database integration</a></h3>
<div class="paragraph">
<p>Use the OPTIONAL <code>characteristics[biosample accession number]</code> column to link samples to <a href="https://www.ebi.ac.uk/biosamples/">BioSamples</a> <a href="#ref5">[5]</a>, enabling cross-database integration with genomics and transcriptomics data. Formats: <code>SAMN*</code> (NCBI) or <code>SAMEA*</code> (EBI).</p>
</div>
</div>
<div class="sect2">
<h3 id="sample-technical-biological-replicates"><a class="anchor" href="#sample-technical-biological-replicates"></a><a class="link" href="#sample-technical-biological-replicates">7.2. Encoding sample technical and biological replicates</a></h3>
<div class="paragraph">
<p>SDRF-Proteomics uses two REQUIRED columns to track replicates <a href="#ref4">[4]</a>:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>characteristics[biological replicate]</strong>: Independent biological samples. Numbering restarts per experimental condition (factor value group).</p>
</li>
<li>
<p><strong>comment[technical replicate]</strong>: Repeated measurements of the same sample (e.g., multiple injections)</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>When no replicates are performed, set both columns to <code>1</code>. For pooled samples, use <code>pooled</code> for biological replicate.</p>
</div>
<div class="sdrf-example-table"><table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
</colgroup>
<thead>
<tr>
<th class="sample-col tableblock halign-left valign-top">source name</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[biological replicate]</th>
<th class="data-col tableblock halign-left valign-top">comment[fraction identifier]</th>
<th class="data-col tableblock halign-left valign-top">comment[technical replicate]</th>
<th class="data-col tableblock halign-left valign-top">comment[data file]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">patient_001</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">P001_F1_TR1.raw</p></td>
</tr>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">patient_001</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">2</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">P001_F1_TR2.raw</p></td>
</tr>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">patient_002</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">2</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">P002_F1_TR1.raw</p></td>
</tr>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">patient_002</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">2</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">2</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">P002_F1_TR2.raw</p></td>
</tr>
</tbody>
</table><div class="sdrf-legend"><div class="legend-item"><span class="legend-color sample"></span>Sample metadata</div><div class="legend-item"><span class="legend-color data"></span>Data file metadata</div></div></div>
</div>
<div class="sect2">
<h3 id="pooled-samples"><a class="anchor" href="#pooled-samples"></a><a class="link" href="#pooled-samples">7.3. Pooled samples</a></h3>
<div class="paragraph">
<p>When multiple samples are pooled into one (e.g., TMT/iTRAQ reference channels for normalization), use the <em>characteristics[pooled sample]</em> column to indicate pooling status. Allowed values:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>not pooled</strong>: Regular individual samples</p>
</li>
<li>
<p><strong>pooled</strong>: Sample is pooled but individual sources are unknown</p>
</li>
<li>
<p><strong>SN=sample1;SN=sample2;&#8230;&#8203;</strong>: Lists source names of pooled samples when known</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Example:</p>
</div>
<div class="sdrf-example-table"><table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 14.2857%;">
<col style="width: 28.5714%;">
<col style="width: 14.2857%;">
<col style="width: 14.2857%;">
<col style="width: 14.2857%;">
<col style="width: 14.2858%;">
</colgroup>
<thead>
<tr>
<th class="sample-col tableblock halign-left valign-top">source name</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[pooled sample]</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[organism]</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[age]</th>
<th class="data-col tableblock halign-left valign-top">comment[label]</th>
<th class="data-col tableblock halign-left valign-top">comment[data file]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">sample_1</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">not pooled</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">homo sapiens</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">45Y</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">TMT126</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">file01.raw</p></td>
</tr>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">sample_2</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">not pooled</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">homo sapiens</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">52Y</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">TMT127N</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">file01.raw</p></td>
</tr>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">pooled_ref</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">SN=sample_1;SN=sample_2</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">homo sapiens</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">pooled</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">TMT131C</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">file01.raw</p></td>
</tr>
</tbody>
</table><div class="sdrf-legend"><div class="legend-item"><span class="legend-color sample"></span>Sample metadata</div><div class="legend-item"><span class="legend-color data"></span>Data file metadata</div></div></div>
<div class="admonitionblock tip">
<table>
<tr>
<td class="icon">
<div class="title">Tip</div>
</td>
<td class="content">
For pooled samples, use <code>pooled</code> for individual-specific fields (biological replicate, age, sex) to indicate a mixture rather than a single sample.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="_sample_metadata_guidelines"><a class="anchor" href="#_sample_metadata_guidelines"></a><a class="link" href="#_sample_metadata_guidelines">7.4. Sample Metadata Guidelines</a></h3>
<div class="paragraph">
<p>For detailed guidance on annotating sample metadata, refer to the following conventions documents:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="metadata-guidelines/sample-metadata.html">Sample Metadata Guidelines</a> - Detailed guidelines for age, sex, disease, organism part, cell type, developmental stage, spiked-in samples, and other sample characteristics</p>
</li>
<li>
<p><a href="templates/human.html">Human Sample Metadata Guidelines</a> - Human-specific metadata including disease staging, treatment history, demographics, and lifestyle factors</p>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="from-sample-data"><a class="anchor" href="#from-sample-data"></a><a class="link" href="#from-sample-data">8. SDRF-Proteomics: data files metadata</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The connection between samples and data files is done using properties annotated with the <code>comment</code> prefix. All properties referring to a data file (e.g., MS run file) are annotated with the category <code>comment</code>. This differentiates data file properties from sample properties (characteristics).</p>
</div>
<div class="sect2">
<h3 id="cv-term-format"><a class="anchor" href="#cv-term-format"></a><a class="link" href="#cv-term-format">8.1. CV Term Format for Data File Metadata</a></h3>
<div class="paragraph">
<p>For data file metadata (<code>comment</code> columns) that reference ontology terms, use the structured format: <code>NT={term name};AC={accession}</code></p>
</div>
<div class="paragraph">
<p>Examples: <code>NT=HCD;AC=PRIDE:0000590</code>, <code>NT=Orbitrap;AC=MS:1000484</code></p>
</div>
<div class="paragraph">
<p>This format enables automated validation and software extraction from raw files. Sample metadata (characteristics) can use simple term names since they are typically human-annotated.</p>
</div>
<div class="paragraph">
<p>The following properties MUST be provided for each data file in <strong>mass spectrometry-based proteomics</strong> experiments. For <strong>affinity-based proteomics</strong> (Olink, SomaScan), see the <a href="templates/affinity-proteomics.html">Affinity-Proteomics template</a> for different required columns.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 30%;">
<col style="width: 10%;">
<col style="width: 40%;">
<col style="width: 20%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Column</th>
<th class="tableblock halign-left valign-top">Requirement</th>
<th class="tableblock halign-left valign-top">Description</th>
<th class="tableblock halign-left valign-top">Ontology</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>assay name</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">REQUIRED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Unique identifier for an MS run/data file</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Free text</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>technology type</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">REQUIRED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Technology used to capture the data</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Fixed values</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>comment[proteomics data acquisition method]</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">REQUIRED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">DDA, DIA, PRM, SRM</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PRIDE:0000659</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>comment[label]</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">REQUIRED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Label applied to sample (or "label free sample")</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PRIDE - Labels</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>comment[instrument]</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">REQUIRED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Mass spectrometer model</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PSI-MS - Instruments</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>comment[cleavage agent details]</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">REQUIRED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Enzyme information (use "not applicable" for top-down/undigested samples)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PSI-MS - Cleavage agents</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>comment[fraction identifier]</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">REQUIRED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Fraction number (1 if not fractionated)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Integer</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>comment[technical replicate]</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">REQUIRED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Technical replicate number (1 if none)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Integer</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>comment[data file]</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">REQUIRED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Name of the raw file</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Free text</p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>Example:</p>
</div>
<div class="sdrf-example-table"><table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 25%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
</colgroup>
<thead>
<tr>
<th class="sample-col tableblock halign-left valign-top">source name</th>
<th class="data-col tableblock halign-left valign-top">assay name</th>
<th class="data-col tableblock halign-left valign-top">technology type</th>
<th class="data-col tableblock halign-left valign-top">comment[proteomics data acquisition method]</th>
<th class="data-col tableblock halign-left valign-top">comment[label]</th>
<th class="data-col tableblock halign-left valign-top">comment[instrument]</th>
<th class="data-col tableblock halign-left valign-top">comment[data file]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">sample_1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">sample1_run1</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">proteomic profiling by mass spectrometry</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">data-dependent acquisition</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">label free sample</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">Q Exactive HF</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">sample1.raw</p></td>
</tr>
</tbody>
</table><div class="sdrf-legend"><div class="legend-item"><span class="legend-color sample"></span>Sample metadata</div><div class="legend-item"><span class="legend-color data"></span>Data file metadata</div></div></div>
</div>
<div class="sect2">
<h3 id="sample-prep-fragmentation"><a class="anchor" href="#sample-prep-fragmentation"></a><a class="link" href="#sample-prep-fragmentation">8.2. Sample Preparation and Fragmentation (MS-based only)</a></h3>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
This section applies to <strong>mass spectrometry-based proteomics</strong> experiments only. For affinity-based proteomics, these properties do not apply.
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>For detailed documentation of sample preparation and MS/MS fragmentation properties, see the <a href="templates/ms-proteomics.html">MS-Proteomics Template</a>:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>Sample preparation</strong>: depletion, reduction reagent, alkylation reagent</p>
</li>
<li>
<p><strong>Fractionation</strong>: fractionation method (used with <code>comment[fraction identifier]</code>)</p>
</li>
<li>
<p><strong>Fragmentation</strong>: collision energy, dissociation method</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="data-acquisition-method"><a class="anchor" href="#data-acquisition-method"></a><a class="link" href="#data-acquisition-method">8.3. Proteomics data acquisition method</a></h3>
<div class="paragraph">
<p>Proteomics data acquisition method can happen in multiple ways: Data Dependent Acquisition (DDA), Data Independent Acquisition (DIA), and targeted approaches. The SDRF-Proteomics file format REQUIRES capturing the method used for the data acquisition in the <em>comment[proteomics data acquisition method]</em> column. The values MUST be children of the PRIDE ontology term <a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000659">proteomics data acquisition method (PRIDE:0000659)</a>. The following values are commonly used:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000627">data-dependent acquisition</a></p>
</li>
<li>
<p><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000450">data-independent acquisition</a></p>
<div class="ulist">
<ul>
<li>
<p><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000650?lang=en">diaPASEF</a></p>
</li>
<li>
<p><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000447">SWATH MS</a></p>
</li>
</ul>
</div>
</li>
<li>
<p><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000629">parallel reaction monitoring</a></p>
</li>
<li>
<p><a href="https://www.ebi.ac.uk/ols4/ontologies/pride/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPRIDE_0000630">selected reaction monitoring</a></p>
</li>
</ul>
</div>
<div class="admonitionblock important">
<table>
<tr>
<td class="icon">
<div class="title">Important</div>
</td>
<td class="content">
The <em>comment[proteomics data acquisition method]</em> column is REQUIRED for all mass spectrometry-based SDRF files. This field must be explicitly specified and cannot be omitted or assumed.
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>You can find an example of a DIA experiment in the following link: <a href="https://github.com/bigbio/proteomics-metadata-standard/blob/master/annotated-projects/PXD018830/PXD018830-DIA.sdrf.tsv">DIA example</a></p>
</div>
<div class="admonitionblock tip">
<table>
<tr>
<td class="icon">
<div class="title">Tip</div>
</td>
<td class="content">
For DIA experiments, additional properties like MS1 scan range can be captured. See <a href="templates/ms-proteomics.html#_dia_scan_window">DIA Scan Window Limits</a> in the MS-Proteomics Template.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="_ms_proteomics_template"><a class="anchor" href="#_ms_proteomics_template"></a><a class="link" href="#_ms_proteomics_template">8.4. MS-Proteomics Template</a></h3>
<div class="paragraph">
<p>For detailed guidance on data file metadata, refer to the conventions document:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="templates/ms-proteomics.html">MS-Proteomics Template</a> - Detailed guidelines for labels, instruments, modifications, cleavage agents, mass tolerances, RAW file URIs, and other data file properties</p>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="additional-sdrf-rules"><a class="anchor" href="#additional-sdrf-rules"></a><a class="link" href="#additional-sdrf-rules">9. Additional SDRF Rules</a></h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="column-cardinality"><a class="anchor" href="#column-cardinality"></a><a class="link" href="#column-cardinality">9.1. Column Cardinality</a></h3>
<div class="paragraph">
<p>Some columns can appear multiple times for the same sample. The cardinality rules are:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>Single (1)</strong>: Column appears exactly once per sample (e.g., <code>characteristics[biological replicate]</code>)</p>
</li>
<li>
<p><strong>Multiple (*)</strong>: Column can appear multiple times (e.g., <code>comment[modification parameters]</code> can specify multiple post-translational modifications)</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Example of multiple <code>comment[modification parameters]</code> columns:</p>
</div>
<div class="sdrf-example-table"><table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 14.2857%;">
<col style="width: 14.2857%;">
<col style="width: 28.5714%;">
<col style="width: 28.5714%;">
<col style="width: 14.2858%;">
</colgroup>
<thead>
<tr>
<th class="sample-col tableblock halign-left valign-top">source name</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[&#8230;&#8203;]</th>
<th class="data-col tableblock halign-left valign-top">comment[modification parameters]</th>
<th class="data-col tableblock halign-left valign-top">comment[modification parameters]</th>
<th class="tableblock halign-left valign-top">&#8230;&#8203;</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">sample-1</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">&#8230;&#8203;</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=fixed;PP=Anywhere</p></td>
<td class="data-col tableblock halign-left valign-top"><p class="tableblock">NT=Oxidation;AC=UNIMOD:35;TA=M;MT=variable;PP=Anywhere</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&#8230;&#8203;</p></td>
</tr>
</tbody>
</table><div class="sdrf-legend"><div class="legend-item"><span class="legend-color sample"></span>Sample metadata</div><div class="legend-item"><span class="legend-color data"></span>Data file metadata</div></div></div>
</div>
<div class="sect2">
<h3 id="row-uniqueness"><a class="anchor" href="#row-uniqueness"></a><a class="link" href="#row-uniqueness">9.2. Row Uniqueness Requirements</a></h3>
<div class="paragraph">
<p>Uniqueness constraints ensure data integrity:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>MUST be unique</strong> (error): <code>source name</code> + <code>assay name</code> + <code>comment[label]</code></p>
</li>
<li>
<p><strong>SHOULD be unique</strong> (warning): <code>source name</code> + <code>assay name</code></p>
</li>
<li>
<p><strong>Assay name</strong>: Each data file MUST have a unique <code>assay name</code></p>
</li>
</ul>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
For multiplexed experiments (TMT, iTRAQ), multiple rows share the same <code>assay name</code> since samples are in one MS run. The <code>comment[label]</code> distinguishes samples within the run.
</td>
</tr>
</table>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="templates"><a class="anchor" href="#templates"></a><a class="link" href="#templates">10. Templates</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>A <strong>template</strong> is a predefined set of metadata columns that ensures consistent annotation for specific experiment types. Templates define REQUIRED, RECOMMENDED, and OPTIONAL columns to make datasets FAIR-compliant.</p>
</div>
<div class="sect2">
<h3 id="_template_architecture"><a class="anchor" href="#_template_architecture"></a><a class="link" href="#_template_architecture">10.1. Template Architecture</a></h3>
<div class="paragraph">
<p>Templates follow a layered hierarchy:</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 33.3333%;">
<col style="width: 33.3333%;">
<col style="width: 33.3334%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Layer</th>
<th class="tableblock halign-left valign-top">Templates</th>
<th class="tableblock halign-left valign-top">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>TECHNOLOGY</strong> (required)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/ms-proteomics.html">ms-proteomics</a>, <a href="templates/affinity-proteomics.html">affinity-proteomics</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Minimum valid SDRF - choose one</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>SAMPLE</strong> (recommended)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/human.html">human</a>, <a href="templates/vertebrates.html">vertebrates</a>, <a href="templates/invertebrates.html">invertebrates</a>, <a href="templates/plants.html">plants</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Organism-specific metadata</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>EXPERIMENT</strong> (optional)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/cell-lines.html">cell-lines</a>, <a href="templates/crosslinking.html">crosslinking</a>, <a href="templates/dda-acquisition.html">dda-acquisition</a>, <a href="templates/dia-acquisition.html">dia-acquisition</a>, <a href="templates/single-cell.html">single-cell</a>, <a href="templates/immunopeptidomics.html">immunopeptidomics</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Methodology-specific columns</p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>Child templates inherit all columns from parents and may add new columns or strengthen requirements (e.g., <code>optional</code> â†’ <code>required</code>).</p>
</div>
</div>
<div class="sect2">
<h3 id="_specifying_templates_in_sdrf_files"><a class="anchor" href="#_specifying_templates_in_sdrf_files"></a><a class="link" href="#_specifying_templates_in_sdrf_files">10.2. Specifying Templates in SDRF Files</a></h3>
<div class="paragraph">
<p>Declare templates using <code>comment[sdrf template]</code> columns. Only list leaf templates (parents are implied). When using multiple templates, add multiple columns with the same name. Two formats are supported:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Simple format (preferred): <code>template_name vX.Y.Z</code></p>
</li>
<li>
<p>Key=value format: <code>NT=template_name;VV=vX.Y.Z</code></p>
</li>
</ul>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlightjs highlight"><code class="language-tsv hljs" data-lang="tsv">source name	...	comment[sdrf template]	comment[sdrf template]
sample_1	...	human v1.1.0	crosslinking v1.0.0</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>Common examples:</strong></p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Experiment Type</th>
<th class="tableblock halign-left valign-top">Template Columns</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Human MS proteomics</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>comment[sdrf template]</code> = <code>human v1.1.0</code></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Mouse MS proteomics</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>comment[sdrf template]</code> = <code>vertebrates v1.1.0</code></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Human crosslinking</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Two columns: <code>human v1.1.0</code> + <code>crosslinking v1.0.0</code></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Human Olink</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Two columns: <code>human v1.1.0</code> + <code>olink v1.0.0</code></p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_available_templates"><a class="anchor" href="#_available_templates"></a><a class="link" href="#_available_templates">10.3. Available Templates</a></h3>
<div class="paragraph">
<p><strong>Sample templates</strong> (organism-specific):</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 33.3333%;">
<col style="width: 33.3333%;">
<col style="width: 33.3334%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Template</th>
<th class="tableblock halign-left valign-top">Use For</th>
<th class="tableblock halign-left valign-top">Key Columns</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/human.html">Human</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Human clinical samples</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">disease, age, sex, ancestry</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/vertebrates.html">Vertebrates</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Mouse, rat, zebrafish</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">disease, developmental stage, strain</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/invertebrates.html">Invertebrates</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Drosophila, C. elegans</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">disease, developmental stage, genotype</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="templates/plants.html">Plants</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Arabidopsis, crops</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">disease, developmental stage, growth conditions</p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p><strong>Experiment-type templates</strong>:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="templates/dda-acquisition.html">DDA Acquisition</a> - dissociation method, collision energy, modifications</p>
</li>
<li>
<p><a href="templates/dia-acquisition.html">DIA Acquisition</a> - scan windows, isolation width, spectral library</p>
</li>
<li>
<p><a href="templates/cell-lines.html">Cell Lines</a> - Cellosaurus integration</p>
</li>
<li>
<p><a href="templates/single-cell.html">Single-Cell</a> - cell isolation, carrier proteome</p>
</li>
<li>
<p><a href="templates/immunopeptidomics.html">Immunopeptidomics</a> - MHC class, HLA typing</p>
</li>
<li>
<p><a href="templates/crosslinking.html">Crosslinking MS</a> - crosslinker reagents</p>
</li>
<li>
<p><a href="templates/metaproteomics.html">Metaproteomics</a> - environmental sample type</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Download templates from the <a href="https://github.com/bigbio/proteomics-metadata-standard/tree/master/sdrf-proteomics/templates">templates folder</a>.</p>
</div>
</div>
<div class="sect2">
<h3 id="_extending_templates"><a class="anchor" href="#_extending_templates"></a><a class="link" href="#_extending_templates">10.4. Extending Templates</a></h3>
<div class="paragraph">
<p>You can add custom columns beyond template requirements for study-specific metadata. Rules:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Use <code>characteristics[&#8230;&#8203;]</code> for sample metadata, <code>comment[&#8230;&#8203;]</code> for technical metadata</p>
</li>
<li>
<p>Column names MUST be valid ontology terms (search <a href="https://www.ebi.ac.uk/ols4/">OLS</a>)</p>
</li>
<li>
<p>Use controlled vocabularies for values when available</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>See <a href="metadata-guidelines/sample-metadata.html#common-additional-columns">Common Additional Columns</a> and <a href="sdrf-terms.html">SDRF Terms Reference</a> for commonly used columns.</p>
</div>
</div>
<div class="sect2">
<h3 id="_contributing_new_templates"><a class="anchor" href="#_contributing_new_templates"></a><a class="link" href="#_contributing_new_templates">10.5. Contributing New Templates</a></h3>
<div class="paragraph">
<p>To propose a new template, open an <a href="https://github.com/bigbio/proteomics-metadata-standard/issues">issue on GitHub</a> and submit a pull request.</p>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_factor_values_study_variables"><a class="anchor" href="#_factor_values_study_variables"></a><a class="link" href="#_factor_values_study_variables">11. Factor Values (Study Variables)</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>Factor values identify the experimental variables being studied - the conditions you want to compare in your analysis. They highlight which sample characteristics are the focus of your experiment.</p>
</div>
<div class="sect2">
<h3 id="_column_format"><a class="anchor" href="#_column_format"></a><a class="link" href="#_column_format">11.1. Column Format</a></h3>
<div class="paragraph">
<p><code>factor value[{variable name}]</code></p>
</div>
</div>
<div class="sect2">
<h3 id="_when_to_use_factor_values"><a class="anchor" href="#_when_to_use_factor_values"></a><a class="link" href="#_when_to_use_factor_values">11.2. When to Use Factor Values</a></h3>
<div class="paragraph">
<p>Use factor values to indicate:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>The primary variable(s) under investigation</p>
</li>
<li>
<p>Conditions being compared (e.g., disease vs. normal, treated vs. untreated)</p>
</li>
<li>
<p>Variables that define experimental groups</p>
</li>
</ul>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
Use <code>normal</code> (not "control") in the disease field for healthy samples. "Control" is an experimental design concept, not a disease state. See <a href="metadata-guidelines/sample-metadata.html#disease-annotation">Disease Annotation Guidelines</a> for details.
</td>
</tr>
</table>
</div>
</div>
<div class="sect2">
<h3 id="_rules"><a class="anchor" href="#_rules"></a><a class="link" href="#_rules">11.3. Rules</a></h3>
<div class="ulist">
<ul>
<li>
<p>Factor value columns SHOULD appear after all characteristics and comment columns</p>
</li>
<li>
<p>Multiple factor values can be used when studying multiple variables</p>
</li>
<li>
<p>The value in a factor value column typically mirrors a characteristics column value</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="_example"><a class="anchor" href="#_example"></a><a class="link" href="#_example">11.4. Example</a></h3>
<div class="paragraph">
<p>In an experiment comparing tumor vs. normal tissue across different cancer stages:</p>
</div>
<div class="sdrf-example-table"><table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 14.2857%;">
<col style="width: 14.2857%;">
<col style="width: 14.2857%;">
<col style="width: 14.2857%;">
<col style="width: 14.2857%;">
<col style="width: 14.2857%;">
<col style="width: 14.2858%;">
</colgroup>
<thead>
<tr>
<th class="sample-col tableblock halign-left valign-top">source name</th>
<th class="tableblock halign-left valign-top">&#8230;&#8203;</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[disease]</th>
<th class="sample-col tableblock halign-left valign-top">characteristics[disease staging]</th>
<th class="tableblock halign-left valign-top">&#8230;&#8203;</th>
<th class="factor-col tableblock halign-left valign-top">factor value[disease]</th>
<th class="factor-col tableblock halign-left valign-top">factor value[disease staging]</th>
</tr>
</thead>
<tbody>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">tumor_sample_1</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&#8230;&#8203;</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">breast carcinoma</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">stage II</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&#8230;&#8203;</p></td>
<td class="factor-col tableblock halign-left valign-top"><p class="tableblock">breast carcinoma</p></td>
<td class="factor-col tableblock halign-left valign-top"><p class="tableblock">stage II</p></td>
</tr>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">normal_sample_1</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&#8230;&#8203;</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">normal</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">not applicable</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&#8230;&#8203;</p></td>
<td class="factor-col tableblock halign-left valign-top"><p class="tableblock">normal</p></td>
<td class="factor-col tableblock halign-left valign-top"><p class="tableblock">not applicable</p></td>
</tr>
<tr>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">tumor_sample_2</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&#8230;&#8203;</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">breast carcinoma</p></td>
<td class="sample-col tableblock halign-left valign-top"><p class="tableblock">stage III</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&#8230;&#8203;</p></td>
<td class="factor-col tableblock halign-left valign-top"><p class="tableblock">breast carcinoma</p></td>
<td class="factor-col tableblock halign-left valign-top"><p class="tableblock">stage III</p></td>
</tr>
</tbody>
</table><div class="sdrf-legend"><div class="legend-item"><span class="legend-color sample"></span>Sample metadata</div><div class="legend-item"><span class="legend-color factor"></span>Factor values</div></div></div>
<div class="paragraph">
<p>In this example, both <code>disease</code> and <code>disease staging</code> are factor values because the experiment aims to compare expression differences between disease states and across cancer stages.</p>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="ontologies-supported"><a class="anchor" href="#ontologies-supported"></a><a class="link" href="#ontologies-supported">12. Ontologies and Controlled Vocabularies</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>SDRF-Proteomics uses ontologies and controlled vocabularies (CVs) to standardize metadata values. The following ontologies are supported:</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 11.1111%;">
<col style="width: 33.3333%;">
<col style="width: 33.3333%;">
<col style="width: 22.2223%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-center valign-top">Category</th>
<th class="tableblock halign-left valign-top">Ontology/CV</th>
<th class="tableblock halign-left valign-top">Description</th>
<th class="tableblock halign-left valign-top">Notes</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-center valign-top" colspan="4"><p class="tableblock"><strong><strong>General Purpose</strong></strong></p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">General</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/efo">Experimental Factor Ontology (EFO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">General experimental metadata</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">General</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/pato">PATO</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Phenotype and Trait Ontology</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">General</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/ncit">NCI Thesaurus (NCIT)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Biomedical terminology</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">General</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/pride">PRIDE Controlled Vocabulary</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Proteomics-specific terms</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top" colspan="4"><p class="tableblock"><strong><strong>Organism and Taxonomy</strong></strong></p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Taxonomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/ncbitaxon">NCBI Taxonomy (NCBITaxon)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Organism classification</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top" colspan="4"><p class="tableblock"><strong><strong>Anatomy and Cell Types</strong></strong></p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/uberon">UBERON</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cross-species anatomy ontology</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Cell Type</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/cl">Cell Ontology (CL)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cell type classification</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/bto">BRENDA Tissue Ontology (BTO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Tissues and cell lines</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/po">Plant Ontology (PO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Plant anatomy and development</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">For plant samples</p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/fbbt">FlyBase Anatomy (FBbt)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Drosophila anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">For Drosophila samples</p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/wbbt">WormBase Anatomy (WBbt)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">C. elegans anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">For C. elegans samples</p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Anatomy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/zfa">Zebrafish Anatomy (ZFA)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Zebrafish anatomy and development</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">For zebrafish samples</p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top" colspan="4"><p class="tableblock"><strong><strong>Disease</strong> (see <a href="metadata-guidelines/sample-metadata.html#disease-annotation">Disease Annotation Guidelines</a>)</strong></p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Disease</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/mondo">Mondo Disease Ontology (MONDO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Unified disease ontology</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">RECOMMENDED</p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Disease</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/efo">Experimental Factor Ontology (EFO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Disease terms from EFO</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Healthy samples</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/pato">Phenotype And Trait Ontology (PATO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Use <code>normal</code> (PATO:0000461) for healthy samples</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top" colspan="4"><p class="tableblock"><strong><strong>Cell Lines</strong></strong></p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Cell Lines</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.cellosaurus.org/">Cellosaurus</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cell line knowledge resource</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">RECOMMENDED</p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Cell Lines</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/clo">Cell Line Ontology (CLO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cell line ontology</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top" colspan="4"><p class="tableblock"><strong><strong>Mass Spectrometry and Proteomics</strong></strong></p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">MS/Proteomics</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/ms">PSI Mass Spectrometry CV (PSI-MS)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Instruments, methods, parameters</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Modifications</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/unimod">Unimod</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Protein modifications database</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Modifications</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/mod">PSI-MOD CV</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Protein modifications ontology</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top" colspan="4"><p class="tableblock"><strong><strong>Other</strong></strong></p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Chemistry</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/chebi">ChEBI</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Chemical Entities of Biological Interest</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Environment</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/envo">Environment Ontology (ENVO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Environmental sample classification</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">For metaproteomics</p></td>
</tr>
<tr>
<td class="tableblock halign-center valign-top"><p class="tableblock">Ancestry</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="https://www.ebi.ac.uk/ols4/ontologies/hancestro">Human Ancestry Ontology (HANCESTRO)</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Human ancestry categories</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">For human samples</p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="sect1">
<h2 id="example-annotated-datasets"><a class="anchor" href="#example-annotated-datasets"></a><a class="link" href="#example-annotated-datasets">13. Examples of Annotated Datasets</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The following table provides links to example SDRF files for different experiment types. Click "View in Explorer" to open the SDRF file in the interactive viewer.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Experiment Type</th>
<th class="tableblock halign-left valign-top">Dataset</th>
<th class="tableblock halign-left valign-top">Description</th>
<th class="tableblock halign-left valign-top">View</th>
<th class="tableblock halign-left valign-top">Source</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Label-free</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PXD008934</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Human proteome label-free quantification</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="sdrf-explorer.html?view=PXD008934">View in Explorer</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="./sdrf-explorer.html?view=PXD008934">GitHub</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">TMT</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PXD017710</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">TMT-labeled quantitative proteomics</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="sdrf-explorer.html?view=PXD017710">View in Explorer</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="./sdrf-explorer.html?view=PXD017710">GitHub</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">SILAC</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PXD000612</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SILAC-based quantification</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="sdrf-explorer.html?view=PXD000612">View in Explorer</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="./sdrf-explorer.html?view=PXD000612">GitHub</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">DIA</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PXD018830</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Data-independent acquisition</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="sdrf-explorer.html?view=PXD018830">View in Explorer</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="./sdrf-explorer.html?view=PXD018830">GitHub</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Phosphoproteomics</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PXD000759</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PTM enrichment study</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="sdrf-explorer.html?view=PXD000759">View in Explorer</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="./sdrf-explorer.html?view=PXD000759">GitHub</a></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cell lines</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PXD001819</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cell line proteomics</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="sdrf-explorer.html?view=PXD001819">View in Explorer</a></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><a href="./sdrf-explorer.html?view=PXD001819">GitHub</a></p></td>
</tr>
</tbody>
</table>
<div class="admonitionblock tip">
<table>
<tr>
<td class="icon">
<div class="title">Tip</div>
</td>
<td class="content">
Use the <a href="sdrf-explorer.html">SDRF Explorer</a> to browse all {total_datasets}+ annotated datasets with filtering, statistics, and interactive viewing.
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>A comprehensive collection of annotated projects is available at: <a href="https://github.com/bigbio/proteomics-metadata-standard/tree/master/annotated-projects">Annotated Projects Repository</a></p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_intellectual_property_statement"><a class="anchor" href="#_intellectual_property_statement"></a><a class="link" href="#_intellectual_property_statement">14. Intellectual Property Statement</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The PSI takes no position regarding the validity or scope of any intellectual property or other rights that might be claimed to pertain to the implementation or use of the technology described in this document or the extent to which any license under such rights might or might not be available; neither does it represent that it has made any effort to identify any such rights. Copies of claims of rights made available for publication and any assurances of licenses to be made available or the result of an attempt made to obtain a general license or permission for the use of such proprietary rights by implementers or users of this specification can be obtained from the PSI Chair.</p>
</div>
<div class="paragraph">
<p>The PSI invites any interested party to bring to its attention any copyrights, patents or patent applications, or other proprietary rights which may cover technology that may be required to practice this recommendation. Please address the information to the PSI Chair (see contacts information at PSI website).</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_copyright_notice"><a class="anchor" href="#_copyright_notice"></a><a class="link" href="#_copyright_notice">15. Copyright Notice</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>Copyright &#169; Proteomics Standards Initiative (2020). All Rights Reserved.</p>
</div>
<div class="paragraph">
<p>This document and translations of it may be copied and furnished to others, and derivative works that comment on or otherwise explain it or assist in its implementation may be prepared, copied, published, and distributed, in whole or in part, without the restriction of any kind, provided that the above copyright notice and this paragraph are included on all such copies and derivative works. However, this document itself may not be modified in any way, such as by removing the copyright notice or references to the PSI or other organizations, except as needed for the purpose of developing Proteomics Recommendations in which case the procedures for copyrights defined in the PSI Document process must be followed, or as required to translate it into languages other than English.</p>
</div>
<div class="paragraph">
<p>The limited permissions granted above are perpetual and will not be revoked by the PSI or its successors or assigns.</p>
</div>
<div class="paragraph">
<p>This document and the information contained herein is provided on an "AS IS" basis and THE PROTEOMICS STANDARDS INITIATIVE DISCLAIMS ALL WARRANTIES, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO ANY WARRANTY THAT THE USE OF THE INFORMATION HEREIN WILL NOT INFRINGE ANY RIGHTS OR ANY IMPLIED WARRANTIES OF MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE."</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_how_to_cite"><a class="anchor" href="#_how_to_cite"></a><a class="link" href="#_how_to_cite">16. How to cite</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>Please cite this document as:</p>
</div>
<div class="paragraph">
<p>Dai C, FÃ¼llgrabe A, Pfeuffer J, Solovyeva EM, Deng J, Moreno P, Kamatchinathan S, Kundu DJ, George N, Fexova S, GrÃ¼ning B, FÃ¶ll MC, Griss J, Vaudel M, Audain E, Locard-Paulet M, Turewicz M, Eisenacher M, Uszkoreit J, Van Den Bossche T, SchwÃ¤mmle V, Webel H, Schulze S, BouyssiÃ© D, Jayaram S, Duggineni VK, Samaras P, Wilhelm M, Choi M, Wang M, Kohlbacher O, Brazma A, Papatheodorou I, Bandeira N, Deutsch EW, VizcaÃ­no JA, Bai M, Sachsenberg T, Levitsky LI, Perez-Riverol Y. A proteomics sample metadata representation for multiomics integration and big data analysis. Nat Commun. 2021 Oct 6;12(1):5854. doi: 10.1038/s41467-021-26111-3. PMID: 34615866; PMCID: PMC8494749. [Manuscript - <a href="https://www.nature.com/articles/s41467-021-26111-3" class="bare">https://www.nature.com/articles/s41467-021-26111-3</a>]</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_references"><a class="anchor" href="#_references"></a><a class="link" href="#_references">References</a></h2>
<div class="sectionbody">
<div class="ulist bibliography">
<ul class="bibliography">
<li>
<p><a id="ref1"></a>[1] Y. Perez-Riverol, S. European Bioinformatics Community for Mass, Toward a Sample Metadata Standard in Public Proteomics Repositories, J Proteome Res 19(10) (2020) 3906-3909. <a href="https://doi.org/10.1021/acs.jproteome.0c00376">doi:10.1021/acs.jproteome.0c00376</a></p>
</li>
<li>
<p><a id="ref2"></a>[2] A. Gonzalez-Beltran, E. Maguire, S.A. Sansone, P. Rocca-Serra, linkedISA: semantic representation of ISA-Tab experimental metadata, BMC Bioinformatics 15 Suppl 14 (2014) S4. <a href="https://doi.org/10.1186/1471-2105-15-S14-S4">doi:10.1186/1471-2105-15-S14-S4</a></p>
</li>
<li>
<p><a id="ref3"></a>[3] T.F. Rayner, P. Rocca-Serra, P.T. Spellman, et al., A simple spreadsheet-based, MIAME-supportive format for microarray data: MAGE-TAB, BMC Bioinformatics 7 (2006) 489. <a href="https://doi.org/10.1186/1471-2105-7-489">doi:10.1186/1471-2105-7-489</a></p>
</li>
<li>
<p><a id="ref4"></a>[4] P. Blainey, M. Krzywinski, N. Altman, Points of significance: replication, Nat Methods 11(9) (2014) 879-80. <a href="https://doi.org/10.1038/nmeth.3091">doi:10.1038/nmeth.3091</a></p>
</li>
<li>
<p><a id="ref5"></a>[5] D. Gupta, I. Liyanage, Y. Perez-Riverol, et al., BioSamples database: the global hub for sample metadata and multi-omics integration, Nucleic Acids Res (2025). <a href="https://doi.org/10.1093/nar/gkaf1133">doi:10.1093/nar/gkaf1133</a></p>
</li>
</ul>
</div>
</div>
</div>
</div>
<div id="footer">
<div id="footer-text">
Last updated 2026-01-31 10:31:24 UTC
</div>
</div>
<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.3/highlight.min.js"></script>
<script>
if (!hljs.initHighlighting.called) {
  hljs.initHighlighting.called = true
  ;[].slice.call(document.querySelectorAll('pre.highlight > code[data-lang]')).forEach(function (el) { hljs.highlightBlock(el) })
}
</script>
</body>
</html>