<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Asciidoctor 2.0.26">
<title>Data Analysis Metadata Guidelines</title>
<link rel="stylesheet" href="./../css/style.css">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.3/styles/github.min.css">
</head>
<body class="has-doc-header book toc2 toc-left">
<header class="doc-header"><div class="doc-header-brand"><a href="../index.html">SDRF-Proteomics</a></div><nav class="doc-header-nav"><a href="../index.html">Home</a><a href="../specification.html">Specification</a><a href="../index.html#metadata-guidelines" class="nav-current">Metadata Guidelines</a><a href="../index.html#templates">Templates</a><a href="../index.html#tools">Tools</a><a href="../sdrf-explorer.html">Explorer</a><a href="../index.html#contributors">Contributors</a><a href="/dev/" class="version-link">Dev Version</a><a href="https://github.com/bigbio/proteomics-metadata-standard" target="_blank">GitHub</a></nav></header>
<div id="header">
<h1>Data Analysis Metadata Guidelines</h1>
<div id="toc" class="toc2">
<div id="toctitle">Table of Contents</div>
<ul class="sectlevel1">
<li><a href="#status">1. Status of this Guideline</a></li>
<li><a href="#abstract">2. Abstract</a></li>
<li><a href="#motivation">3. Motivation</a>
<ul class="sectlevel2">
<li><a href="#_the_reproducibility_challenge">3.1. The Reproducibility Challenge</a></li>
<li><a href="#_objectives">3.2. Objectives</a></li>
<li><a href="#_development_context">3.3. Development Context</a></li>
</ul>
</li>
<li><a href="#use-cases">4. Use Case Scenarios</a>
<ul class="sectlevel2">
<li><a href="#_reproducibility_of_data_analysis">4.1. Reproducibility of Data Analysis</a></li>
<li><a href="#_transparent_re_processing_of_public_datasets">4.2. Transparent Re-processing of Public Datasets</a></li>
<li><a href="#_workflow_integration">4.3. Workflow Integration</a></li>
</ul>
</li>
<li><a href="#specification">5. Parameter Specification</a></li>
<li><a href="#parameters">6. Parameter Categories</a>
<ul class="sectlevel2">
<li><a href="#identification-params">6.1. Peptide and Protein Identification Parameters</a></li>
<li><a href="#quantification-params">6.2. Peptide and Protein Quantification Parameters</a></li>
<li><a href="#statistics-params">6.3. Statistical Testing Parameters</a></li>
</ul>
</li>
<li><a href="#implementation">7. Implementation Guidelines</a>
<ul class="sectlevel2">
<li><a href="#_parameter_values_and_types">7.1. Parameter Values and Types</a>
<ul class="sectlevel3">
<li><a href="#_example_parameter_definition">7.1.1. Example Parameter Definition</a></li>
</ul>
</li>
<li><a href="#_usage_in_sdrf_files">7.2. Usage in SDRF Files</a></li>
<li><a href="#_example_implementation">7.3. Example Implementation</a></li>
</ul>
</li>
<li><a href="#validation">8. Validation and Quality Control</a>
<ul class="sectlevel2">
<li><a href="#_required_vs_optional_parameters">8.1. Required vs. Optional Parameters</a></li>
<li><a href="#_validation_tools">8.2. Validation Tools</a></li>
</ul>
</li>
<li><a href="#extension-guidelines">9. Extension and Maintenance</a>
<ul class="sectlevel2">
<li><a href="#_adding_new_parameters">9.1. Adding New Parameters</a></li>
<li><a href="#_version_management">9.2. Version Management</a></li>
</ul>
</li>
<li><a href="#authors">10. Authors and Contributors</a></li>
<li><a href="#future-directions">11. Future Directions</a>
<ul class="sectlevel2">
<li><a href="#_planned_extensions">11.1. Planned Extensions</a></li>
<li><a href="#_community_feedback">11.2. Community Feedback</a></li>
</ul>
</li>
<li><a href="#references">12. References</a></li>
</ul>
</div>
</div>
<div id="content">
<div class="sect1">
<h2 id="status"><a class="anchor" href="#status"></a><a class="link" href="#status">1. Status of this Guideline</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>This document provides information about the data analysis metadata guidelines for SDRF-Proteomics. These guidelines extend the core SDRF-Proteomics specification to capture computational analysis parameters that are essential for reproducibility and re-analysis of proteomics datasets.</p>
</div>
<div class="paragraph">
<p><strong>Version 1.0.0</strong> - 2024-01-15</p>
</div>
<div class="paragraph">
<p><strong>Scope</strong>: Bottom-up LC-MS proteomics experiments without labeling</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="abstract"><a class="anchor" href="#abstract"></a><a class="link" href="#abstract">2. Abstract</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The Data Analysis Metadata Guidelines define standardized parameters to capture computational analysis settings in SDRF-Proteomics files. These guidelines address the critical need for reproducible data analysis by providing machine-readable descriptions of key analysis parameters, enabling transparent re-processing and validation of proteomics results.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="motivation"><a class="anchor" href="#motivation"></a><a class="link" href="#motivation">3. Motivation</a></h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="_the_reproducibility_challenge"><a class="anchor" href="#_the_reproducibility_challenge"></a><a class="link" href="#_the_reproducibility_challenge">3.1. The Reproducibility Challenge</a></h3>
<div class="paragraph">
<p>Differently parametrized data analysis and usage of different software tools and algorithms often lead to substantially different results, creating significant challenges for the reproducibility of proteomics research findings.</p>
</div>
</div>
<div class="sect2">
<h3 id="_objectives"><a class="anchor" href="#_objectives"></a><a class="link" href="#_objectives">3.2. Objectives</a></h3>
<div class="paragraph">
<p>These guidelines tackle these challenges by:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>Standardizing Parameter Documentation</strong>: Adding a set of generalized parameters to the SDRF file format to facilitate reproducing data analysis results</p>
</li>
<li>
<p><strong>Enabling Cross-Platform Analysis</strong>: Providing standardized parameter descriptions that can be interpreted across different analysis pipelines</p>
</li>
<li>
<p><strong>Supporting Workflow Standardization</strong>: Contributing to the standardization of data analysis workflows and their settings</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="_development_context"><a class="anchor" href="#_development_context"></a><a class="link" href="#_development_context">3.3. Development Context</a></h3>
<div class="paragraph">
<p>The collection and definition of these parameters has been developed as part of the ELIXIR Implementation Study and are currently focused on standard bottom-up LC-MS proteomics experiments without labeling.</p>
</div>
<div class="admonitionblock important">
<table>
<tr>
<td class="icon">
<div class="title">Important</div>
</td>
<td class="content">
We acknowledge potential challenges with the transferability of data analysis parameters across different pipelines. This collection serves as a foundation for further standardization efforts in proteomics data analysis workflows.
</td>
</tr>
</table>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="use-cases"><a class="anchor" href="#use-cases"></a><a class="link" href="#use-cases">4. Use Case Scenarios</a></h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="_reproducibility_of_data_analysis"><a class="anchor" href="#_reproducibility_of_data_analysis"></a><a class="link" href="#_reproducibility_of_data_analysis">4.1. Reproducibility of Data Analysis</a></h3>
<div class="paragraph">
<p>The standardized parameters in the SDRF file make the main features of data analysis transparent and machine-readable, enabling:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Verification of analysis methods</p>
</li>
<li>
<p>Assessment of parameter choices</p>
</li>
<li>
<p>Comparison across studies</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="_transparent_re_processing_of_public_datasets"><a class="anchor" href="#_transparent_re_processing_of_public_datasets"></a><a class="link" href="#_transparent_re_processing_of_public_datasets">4.2. Transparent Re-processing of Public Datasets</a></h3>
<div class="paragraph">
<p>By documenting key analysis parameters, re-analysis with different software tools or modified parameter values becomes more accessible and reliable.</p>
</div>
</div>
<div class="sect2">
<h3 id="_workflow_integration"><a class="anchor" href="#_workflow_integration"></a><a class="link" href="#_workflow_integration">4.3. Workflow Integration</a></h3>
<div class="paragraph">
<p>Several computational workflows already benefit from this additional information:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="https://github.com/bigbio/quantms">quantms</a></p>
</li>
<li>
<p><a href="https://github.com/wombat-p">WOMBAT-P</a></p>
</li>
<li>
<p>Other SDRF-compatible analysis pipelines</p>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="specification"><a class="anchor" href="#specification"></a><a class="link" href="#specification">5. Parameter Specification</a></h2>
<div class="sectionbody">
<div class="imageblock">
<div class="content">
<img src="assets/SDRF_extended.svg" alt="SDRF Extended Diagram" width="640" height="480">
</div>
</div>
<div class="paragraph">
<p><strong>Figure 1</strong>: Extended SDRF structure incorporating data analysis metadata parameters.</p>
</div>
<div class="paragraph">
<p>All parameters, their types, and allowed values are formally specified in <a href="assets/param2sdrf.yml">param2sdrf.yml</a>. This specification file provides the authoritative definition for implementing these parameters in analysis pipelines.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="parameters"><a class="anchor" href="#parameters"></a><a class="link" href="#parameters">6. Parameter Categories</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p>The data analysis parameters are organized into three main categories reflecting the typical proteomics analysis workflow:</p>
</div>
<div class="sect2">
<h3 id="identification-params"><a class="anchor" href="#identification-params"></a><a class="link" href="#identification-params">6.1. Peptide and Protein Identification Parameters</a></h3>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
Core identification parameters like variable and fixed modifications, precursor and fragment tolerances, and enzyme specifications are already part of the main SDRF-Proteomics specification and are not redefined here.
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>The following parameters are commonly used in database search engines and may be transferable to other identification approaches such as <em>de novo</em> sequencing:</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 16.6666%;">
<col style="width: 33.3333%;">
<col style="width: 50.0001%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Short Name</th>
<th class="tableblock halign-left valign-top">SDRF Column Name</th>
<th class="tableblock halign-left valign-top">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">fions</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[forward ions]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">N-terminal fragment ions used in scoring</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">rions</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[reverse ions]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">C-terminal fragment ions used in scoring</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">isotope_error_range</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[isotope error range]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Allowed isotope offsets of the precursor mass</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">add_decoys</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[add decoys]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Whether decoy sequences have been added to the database</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">num_hits</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[num peptide hits]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Number of peptide hits reported per spectrum by the search engine</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">allowed_miscleavages</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[allowed miscleavages]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Maximum number of allowed missed cleavage sites</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">min_precursor_charge</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[minimum precursor charge]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Lowest allowed precursor charge state</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">max_precursor_charge</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[maximum precursor charge]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Highest allowed precursor charge state</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">min_peptide_length</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[minimum peptide length]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Minimum considered peptide length in amino acids</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">max_peptide_length</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[maximum peptide length]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Maximum considered peptide length in amino acids</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">max_mods</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[maximum allowed modifications]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Maximum number of modifications allowed per peptide</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">ident_fdr_psm</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[fdr on psm level]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">False discovery rate threshold for peptide-spectrum matches</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">ident_fdr_peptide</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[fdr on peptide level]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">False discovery rate threshold for peptide identifications</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">ident_fdr_protein</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[fdr on protein level]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">False discovery rate threshold for protein identifications</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="quantification-params"><a class="anchor" href="#quantification-params"></a><a class="link" href="#quantification-params">6.2. Peptide and Protein Quantification Parameters</a></h3>
<div class="paragraph">
<p>These parameters describe the quantification strategy and processing steps:</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 16.6666%;">
<col style="width: 33.3333%;">
<col style="width: 50.0001%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Short Name</th>
<th class="tableblock halign-left valign-top">SDRF Column Name</th>
<th class="tableblock halign-left valign-top">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">enable_match_between_runs</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[run match between runs]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Whether match-between-runs feature alignment was performed</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">protein_inference</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[protein inference method]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Algorithm or method used to infer proteins from peptide identifications</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">quantification_method</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[quantification method]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Quantification approach (intensity-based, precursor area, MS2-based)</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">summarization_peptides</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[summarization to peptides method]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Method used to summarize PSM values to peptide level</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">summarization_proteins</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[summarization to proteins method]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Method used to summarize peptide values to protein level</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">min_num_peptides</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[minimum number of peptides per protein]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Minimum number of unique peptides required for protein quantification</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">quant_transformation</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[transformation of quantitative values]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Mathematical transformation applied to quantitative values (e.g., log2)</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">normalization_method</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[normalization method]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Method used to normalize samples and remove systematic bias</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="statistics-params"><a class="anchor" href="#statistics-params"></a><a class="link" href="#statistics-params">6.3. Statistical Testing Parameters</a></h3>
<div class="paragraph">
<p>Parameters for differential analysis and statistical testing:</p>
</div>
<div class="admonitionblock note">
<table>
<tr>
<td class="icon">
<div class="title">Note</div>
</td>
<td class="content">
Detailed experimental design information (contrasts, batches) should be readable from the experimental design as captured in the main SDRF structure.
</td>
</tr>
</table>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 16.6666%;">
<col style="width: 33.3333%;">
<col style="width: 50.0001%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Short Name</th>
<th class="tableblock halign-left valign-top">SDRF Column Name</th>
<th class="tableblock halign-left valign-top">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">run_statistics</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[run statistical tests]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Whether statistical analysis was performed to identify differentially regulated features</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">quant_fdr_method</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[method for correction of multiple testing]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Method used for multiple testing correction (e.g., Benjamini-Hochberg)</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">quant_fdr_threshold</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">comment[threshold for statistical test fdr]</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">FDR threshold used to determine significantly differentially regulated features</p></td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="sect1">
<h2 id="implementation"><a class="anchor" href="#implementation"></a><a class="link" href="#implementation">7. Implementation Guidelines</a></h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="_parameter_values_and_types"><a class="anchor" href="#_parameter_values_and_types"></a><a class="link" href="#_parameter_values_and_types">7.1. Parameter Values and Types</a></h3>
<div class="paragraph">
<p>All parameter specifications are maintained in the <code>params2sdrf.yml</code> file, which defines:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Parameter names and SDRF column mappings</p>
</li>
<li>
<p>Data types (boolean, integer, float, string, class)</p>
</li>
<li>
<p>Allowed values for controlled vocabularies</p>
</li>
<li>
<p>Default values where applicable</p>
</li>
</ul>
</div>
<div class="sect3">
<h4 id="_example_parameter_definition"><a class="anchor" href="#_example_parameter_definition"></a><a class="link" href="#_example_parameter_definition">7.1.1. Example Parameter Definition</a></h4>
<div class="listingblock">
<div class="content">
<pre class="highlightjs highlight"><code class="language-yaml hljs" data-lang="yaml">- name: protein_inference
  type: class
  sdrf: protein inference method
  default: unique
  value:
    - unique
    - shared
    - parsimonious
    - other</code></pre>
</div>
</div>
<div class="paragraph">
<p>The <code>type</code> field specifies the parameter data type:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>boolean</strong>: True/false values</p>
</li>
<li>
<p><strong>integer</strong>: Whole numbers</p>
</li>
<li>
<p><strong>float</strong>: Decimal numbers</p>
</li>
<li>
<p><strong>string</strong>: Free text (use sparingly)</p>
</li>
<li>
<p><strong>class</strong>: Controlled vocabulary with predefined values</p>
</li>
</ul>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_usage_in_sdrf_files"><a class="anchor" href="#_usage_in_sdrf_files"></a><a class="link" href="#_usage_in_sdrf_files">7.2. Usage in SDRF Files</a></h3>
<div class="paragraph">
<p>These parameters should be included as additional columns in SDRF files following the standard <code>comment[parameter name]</code> format. They typically appear after the core data file metadata columns.</p>
</div>
</div>
<div class="sect2">
<h3 id="_example_implementation"><a class="anchor" href="#_example_implementation"></a><a class="link" href="#_example_implementation">7.3. Example Implementation</a></h3>
<div class="paragraph">
<p>For a complete example of an SDRF file incorporating these metadata parameters, see <a href="assets/sdrf_local.tsv">sdrf_local.tsv</a>.</p>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="validation"><a class="anchor" href="#validation"></a><a class="link" href="#validation">8. Validation and Quality Control</a></h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="_required_vs_optional_parameters"><a class="anchor" href="#_required_vs_optional_parameters"></a><a class="link" href="#_required_vs_optional_parameters">8.1. Required vs. Optional Parameters</a></h3>
<div class="ulist">
<ul>
<li>
<p><strong>Core parameters</strong>: Essential for basic reproducibility (FDR thresholds, quantification method)</p>
</li>
<li>
<p><strong>Recommended parameters</strong>: Important for comprehensive analysis documentation</p>
</li>
<li>
<p><strong>Optional parameters</strong>: Additional details that may be workflow-specific</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="_validation_tools"><a class="anchor" href="#_validation_tools"></a><a class="link" href="#_validation_tools">8.2. Validation Tools</a></h3>
<div class="paragraph">
<p>Tools supporting these guidelines should validate:</p>
</div>
<div class="olist arabic">
<ol class="arabic">
<li>
<p>Parameter names match the specification</p>
</li>
<li>
<p>Values conform to allowed ranges/vocabularies</p>
</li>
<li>
<p>Required parameters are present</p>
</li>
<li>
<p>Parameter combinations are logically consistent</p>
</li>
</ol>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="extension-guidelines"><a class="anchor" href="#extension-guidelines"></a><a class="link" href="#extension-guidelines">9. Extension and Maintenance</a></h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="_adding_new_parameters"><a class="anchor" href="#_adding_new_parameters"></a><a class="link" href="#_adding_new_parameters">9.1. Adding New Parameters</a></h3>
<div class="paragraph">
<p>New parameters should be:</p>
</div>
<div class="olist arabic">
<ol class="arabic">
<li>
<p>Proposed through GitHub issues or pull requests</p>
</li>
<li>
<p>Reviewed by the community for relevance and clarity</p>
</li>
<li>
<p>Added to the <code>params2sdrf.yml</code> specification</p>
</li>
<li>
<p>Documented with clear descriptions and allowed values</p>
</li>
</ol>
</div>
</div>
<div class="sect2">
<h3 id="_version_management"><a class="anchor" href="#_version_management"></a><a class="link" href="#_version_management">9.2. Version Management</a></h3>
<div class="paragraph">
<p>Changes to parameter specifications follow semantic versioning:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>Major</strong>: Breaking changes to parameter definitions</p>
</li>
<li>
<p><strong>Minor</strong>: New parameters or non-breaking value additions</p>
</li>
<li>
<p><strong>Patch</strong>: Documentation fixes and clarifications</p>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="authors"><a class="anchor" href="#authors"></a><a class="link" href="#authors">10. Authors and Contributors</a></h2>
<div class="sectionbody">
<div class="paragraph">
<p><strong>Primary Authors:</strong>
- Veit Schwämmle
- David Bouyssié</p>
</div>
<div class="paragraph">
<p><strong>Contributing Organizations:</strong>
- ELIXIR Implementation Study participants</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="future-directions"><a class="anchor" href="#future-directions"></a><a class="link" href="#future-directions">11. Future Directions</a></h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="_planned_extensions"><a class="anchor" href="#_planned_extensions"></a><a class="link" href="#_planned_extensions">11.1. Planned Extensions</a></h3>
<div class="ulist">
<ul>
<li>
<p><strong>Labeled Quantification</strong>: Parameters for TMT, SILAC, and other labeling strategies</p>
</li>
<li>
<p><strong>DIA Analysis</strong>: Data-independent acquisition specific parameters</p>
</li>
<li>
<p><strong>Cross-Linking</strong>: Parameters for cross-linking mass spectrometry</p>
</li>
<li>
<p><strong>Top-Down Proteomics</strong>: Intact protein analysis parameters</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="_community_feedback"><a class="anchor" href="#_community_feedback"></a><a class="link" href="#_community_feedback">11.2. Community Feedback</a></h3>
<div class="paragraph">
<p>We encourage community feedback on:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Additional parameters needed for specific workflows</p>
</li>
<li>
<p>Clarification of existing parameter definitions</p>
</li>
<li>
<p>Use cases not currently covered</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Please contribute through the <a href="https://github.com/bigbio/proteomics-metadata-standard">GitHub repository</a>.</p>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="references"><a class="anchor" href="#references"></a><a class="link" href="#references">12. References</a></h2>
<div class="sectionbody">
<div class="ulist">
<ul>
<li>
<p>ELIXIR Implementation Study documentation</p>
</li>
<li>
<p>SDRF-Proteomics core specification</p>
</li>
<li>
<p>Community feedback and use cases</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>For the most current parameter specifications and examples, refer to the associated files in this directory and the main SDRF-Proteomics repository.</p>
</div>
</div>
</div>
</div>
<div id="footer">
<div id="footer-text">
Last updated 2026-01-27 13:36:17 UTC
</div>
</div>
<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.3/highlight.min.js"></script>
<script>
if (!hljs.initHighlighting.called) {
  hljs.initHighlighting.called = true
  ;[].slice.call(document.querySelectorAll('pre.highlight > code[data-lang]')).forEach(function (el) { hljs.highlightBlock(el) })
}
</script>
</body>
</html>